<!DOCTYPE html>
<html lang="en">

  <!-- Head -->
  <head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">    <!-- Metadata, OpenGraph and Schema.org -->
    

    <!-- Standard metadata -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <title>publications | Lab of Computational Neuroscience</title>
    <meta name="author" content="Lab of Computational Neuroscience  ">
    <meta name="description" content="Computational Neuroscience + Applied Mathematics. Research group headed by Douglas Zhou, PhD in the  Institute of Natural Science, Shanghai Jiao Tong University.
">
    <meta name="keywords" content="computational-neuroscience, applied-mathematics">


    <!-- Bootstrap & MDB -->
    <link href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap/4.6.1/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous">

    <!-- Fonts & Icons -->
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous">
    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">

    <!-- Code Syntax Highlighting -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="" id="highlight_theme_light">

    <!-- Styles -->
    
    <link rel="stylesheet" href="/assets/css/main.css">
    <link rel="canonical" href="https://lcns-sjtu.github.io/publications/">
    
    <!-- Dark Mode -->
    
    <link rel="stylesheet" href="https://combinatronics.io/jwarby/pygments-css/master/native.css" media="none" id="highlight_theme_dark">

    <script src="/assets/js/theme.js"></script>
    <script src="/assets/js/dark_mode.js"></script>
    

  </head>

  <!-- Body -->
  <body class="fixed-top-nav ">

    <!-- Header -->
    <header>

      <meta name="theme-color" content="var(--global-theme-color)" media="(prefers-color-scheme: light)">
      <meta name="theme-color" content="var(--global-theme-color)" media="(prefers-color-scheme: dark)">
      <!-- TODO: issue to fix! updating theme-color requires page reloading. -->
      <!-- Nav Bar -->
      <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
        <div class="container">
          <a class="navbar-brand title font-weight-lighter" href="https://lcns-sjtu.github.io/">
              <!--Lab of Computational Neuroscience-->
              <img class="lab-logo">
          </a>
          <!-- Navbar Toggle -->
          <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar top-bar"></span>
            <span class="icon-bar middle-bar"></span>
            <span class="icon-bar bottom-bar"></span>
          </button>

          <div class="collapse navbar-collapse text-right" id="navbarNav">
            <ul class="navbar-nav ml-auto flex-nowrap">

              <!-- About -->
              <li class="nav-item ">
                <a class="nav-link" href="/">about</a>
              </li>

              <!-- Other pages -->
              <li class="nav-item ">
                <a class="nav-link" href="/people/">people</a>
              </li>
              <li class="nav-item active">
                <a class="nav-link" href="/publications/">publications<span class="sr-only">(current)</span></a>
              </li>
              <li class="nav-item ">
                <a class="nav-link" href="/research/">research</a>
              </li>
              <li class="nav-item ">
                <a class="nav-link" href="/recruitment/">recruitment</a>
              </li>
              <li class="nav-item ">
                <a class="nav-link" href="/repositories/">repositories</a>
              </li>
              <!-- Blog -->
              <li class="nav-item ">
                <a class="nav-link" href="/blog/" style="white-space: nowrap;">journal club</a>
              </li>

              <!-- Toogle theme mode -->
              <li class="toggle-container">
                <button id="light-toggle" title="Change theme">
                  <i class="fas fa-moon"></i>
                  <i class="fas fa-sun"></i>
                </button>
              </li>
            </ul>
          </div>
        </div>
      </nav>

      <!-- Scrolling Progress Bar -->
      <progress id="progress" value="0">
        <div class="progress-container">
          <span class="progress-bar"></span>
        </div>
      </progress>
    </header>


    <!-- Content -->
    
      
      <div class="page-content">
        <div class="header-background">
          <div class="img"></div>
          <!-- <div class="text"><h2>Lab of Computational Neuroscience @SJTU</h2></div> -->
        </div>
      </div>
    
    <div class="container mt-5">
      <!-- page.html -->
        <div class="post">

          <header class="post-header">
            
          </header>

          <article>
            <!-- _pages/publications.md -->
<div class="publications">
  <h2 class="year">2024</h2>
  <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="tian2024causal" class="col-sm-11">
        <!-- Title -->
        <div class="title">Causal Connectivity Measures for Pulse-Output Network Reconstruction: Analysis and Applications</div>
        <!-- Author -->
        <div class="author">
        

        <a href="https://lcns-sjtu.github.io/people/zhongqi_tian/">Zhong-qi K. Tian</a>, Chen, Kai, , <a href="https://ins.sjtu.edu.cn/people/songtingli/" rel="external nofollow noopener" target="_blank">Songting Li</a>, and
          <span class="more-authors" title="click to view 2 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '2 more authors' ? 'David W. McLaughlin, Douglas Zhou' : '2 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">2 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Proceedings of the National Academy of Sciences</em>, 121(14), e2305297121 2024
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://www.pnas.org/doi/10.1073/pnas.2305297121" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
            <a href="/assets/pdf/Tian_Chen_Li_McLaughlin_Zhou_PNAS_2024.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1073/pnas.2305297121"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1073/pnas.2305297121" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>The causal connectivity of a network is often inferred to understand network function. It is arguably acknowledged that the inferred causal connectivity relies on the causality measure one applies, and it may differ from the network’s underlying structural connectivity. However, the interpretation of causal connectivity remains to be fully clarified, in particular, how causal connectivity depends on causality measures and how causal connectivity relates to structural connectivity. Here, we focus on nonlinear networks with pulse signals as measured output, e.g., neural networks with spike output, and address the above issues based on four commonly utilized causality measures, i.e., time-delayed correlation coefficient, time-delayed mutual information, Granger causality, and transfer entropy. We theoretically show how these causality measures are related to one another when applied to pulse signals. Taking a simulated Hodgkin–Huxley network and a real mouse brain network as two illustrative examples, we further verify the quantitative relations among the four causality measures and demonstrate that the causal connectivity inferred by any of the four well coincides with the underlying network structural connectivity, therefore illustrating a direct link between the causal and structural connectivity. We stress that the structural connectivity of pulse-output networks can be reconstructed pairwise without conditioning on the global information of all other nodes in a network, thus circumventing the curse of dimensionality. Our framework provides a practical and effective approach for pulse-output network reconstruction.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">tian2024causal</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Causal Connectivity Measures for Pulse-Output Network Reconstruction: {{Analysis}} and Applications}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Tian, Zhong-qi K. and Chen, Kai and Li, Songting and McLaughlin, David W. and Zhou, Douglas}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">apr</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Proceedings of the National Academy of Sciences}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{121}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{14}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{e2305297121}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0027-8424, 1091-6490}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1073/pnas.2305297121}</span><span class="p">,</span>
  <span class="na">copyright</span> <span class="p">=</span> <span class="s">{All rights reserved}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li></ol>

  <h2 class="year">2023</h2>
  <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"><abbr class="badge">CPAM</abbr></div> -->

        <!-- Entry bib key -->
        <div id="li2023mathematical" class="col-sm-11">
        <!-- Title -->
        <div class="title">Mathematical Modeling and Analysis of Spatial Neuron Dynamics: Dendritic Integration and Beyond</div>
        <!-- Author -->
        <div class="author">
        

        <a href="https://ins.sjtu.edu.cn/people/songtingli/" rel="external nofollow noopener" target="_blank">Songting Li</a>, McLaughlin, David W., , and <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Communications on Pure and Applied Mathematics</em>, 76(1), 114–162 2023
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://onlinelibrary.wiley.com/doi/10.1002/cpa.22020" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1002/cpa.22020"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1002/cpa.22020" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">li2023mathematical</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Mathematical {{Modeling}} and {{Analysis}} of {{Spatial Neuron Dynamics}}: {{Dendritic Integration}} and {{Beyond}}}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Li, Songting and McLaughlin, David W. and Zhou, Douglas}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jan</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Communications on Pure and Applied Mathematics}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{76}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{114--162}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0010-3640, 1097-0312}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1002/cpa.22020}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li></ol>

  <h2 class="year">2022</h2>
  <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="MariustHart2022" class="col-sm-11">
        <!-- Title -->
        <div class="title">Neuromatch Academy: a 3-week, online summer school in computational neuroscience</div>
        <!-- Author -->
        <div class="author">
        

        Marius’t Hart, Bernard, ..., , <a href="https://ins.sjtu.edu.cn/people/songtingli/" rel="external nofollow noopener" target="_blank">Songting Li</a>, 
                ..., Uquillas, Federico, and
          <span class="more-authors" title="click to view 1 more author" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '1 more author' ? 'Tara Viegen' : '1 more author';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">1 more author</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>J. Open Source Educ.</em>, 5(49), 118 2022
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://jose.theoj.org/papers/10.21105/jose.00118" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.21105/jose.00118"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.21105/jose.00118" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Neuromatch Academy (https://academy.neuromatch.io; (van Viegen et al., 2021)) was designed as an online summer school to cover the basics of computational neuroscience in three weeks. The materials cover dominant and emerging computational neuroscience tools, how they complement one another, and specifically focus on how they can help us to better understand how the brain functions. An original component of the materials is its focus on modeling choices, i.e. how do we choose the right approach, how do we build models, and how can we evaluate models to determine if they provide real (meaningful) insight. This meta-modeling component of the instructional materials asks what questions can be answered by different techniques, and how to apply them meaningfully to get insight about brain function.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Li2022" class="col-sm-11">
        <!-- Title -->
        <div class="title">Hierarchical timescales in the neocortex: Mathematical mechanism and biological insights</div>
        <!-- Author -->
        <div class="author">
        

        <a href="https://ins.sjtu.edu.cn/people/songtingli/" rel="external nofollow noopener" target="_blank">Songting Li</a>, and  and 
                Wang, Xiao-Jing</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Proc. Natl. Acad. Sci.</em>, 119(6), 118 2022
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://pnas.org/doi/full/10.1073/pnas.2110274119" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1073/pnas.2110274119"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1073/pnas.2110274119" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>In the neocortex, while early sensory areas encode and process external inputs rapidly, higher-association areas are endowed with slow dynamics suitable for accumulating information over time. Such a hierarchy of temporal response windows along the cortical hierarchy naturally emerges in a model of multiareal primate cortex. This finding raises the question of why diverse temporal modes are not mixed in roughly the same way across the whole cortex, despite high connection density and an abundance of feedback loops. We investigate this question by mathematically analyzing the anatomically based network model of macaque cortex and theoretically show that three sufficient conditions of synaptic excitation and inhibition give rise to timescale segregation in a hierarchy, a functionally important characteristic of the cortex.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Li2022</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Li, Songting and Wang, Xiao-Jing}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1073/pnas.2110274119}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0027-8424}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Proc. Natl. Acad. Sci.}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Detailed excitation–inhibition balance of long-ran,Eigenvector localization,Interareal heterogeneity,Large-scale cortical network,Timescale hierarchy}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">feb</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{6}</span><span class="p">,</span>
  <span class="na">pmid</span> <span class="p">=</span> <span class="s">{35110401}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Hierarchical timescales in the neocortex: Mathematical mechanism and biological insights}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{119}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Qian2022111111" class="col-sm-11">
        <!-- Title -->
        <div class="title">A striatal SOM-driven ChAT-iMSN loop generates beta oscillations and produces motor deficits</div>
        <!-- Author -->
        <div class="author">
        

        Qian, Dandan, , Li, Wei, , 
                Xue, Jinwen, and
          <span class="more-authors" title="click to view 13 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '13 more authors' ? 'Yi Wu, Ziling Wang, Tao Shi, Songting Li, Jingxuan Yang, Shi Qiu, Shaoli Wang, Yousheng Shu, Liang Chen, Qiao Wang, Ti-Fei Yuan, Douglas Zhou, Wei Lu' : '13 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">13 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Cell Reports</em>, 40(3), 111111 2022
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://www.sciencedirect.com/science/article/pii/S2211124722009172" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="https://doi.org/10.1016/j.celrep.2022.111111"></span>
              <span class="__dimensions_badge_embed__" data-doi="https://doi.org/10.1016/j.celrep.2022.111111" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Summary
Enhanced beta oscillations within the cortico-basal ganglia-thalamic (CBT) network are correlated with motor deficits in Parkinson’s disease (PD), whose generation has been associated recently with amplified network dynamics in the striatum. However, how distinct striatal cell subtypes interact to orchestrate beta oscillations remains largely unknown. Here, we show that optogenetic suppression of dopaminergic control over the dorsal striatum (DS) elevates the power of local field potentials (LFPs) selectively at beta band (12–25 Hz), accompanied by impairments in locomotion. The amplified beta power originates from a striatal loop driven by somatostatin-expressing (SOM) interneurons and constituted by choline acetyltransferase (ChAT)-expressing interneurons and dopamine D2 receptor (D2R)-expressing medium spiny neurons (iMSNs). Moreover, closed-loop intervention selectively targeting striatal iMSNs or ChATs diminishes beta oscillations and restores motor function. Thus, we reveal a striatal microcircuit motif that underlies beta oscillation generation and accompanied motor deficits upon perturbation of dopaminergic control over the striatum.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Qian2022111111</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{A striatal SOM-driven ChAT-iMSN loop generates beta oscillations and produces motor deficits}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Cell Reports}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{40}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{3}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{111111}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2211-1247}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1016/j.celrep.2022.111111}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Qian, Dandan and Li, Wei and Xue, Jinwen and Wu, Yi and Wang, Ziling and Shi, Tao and Li, Songting and Yang, Jingxuan and Qiu, Shi and Wang, Shaoli and Shu, Yousheng and Chen, Liang and Wang, Qiao and Yuan, Ti-Fei and Zhou, Douglas and Lu, Wei}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{striatum, SNc, SOM cells, ChAT cells, beta oscillations}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
</ol>

  <h2 class="year">2021</h2>
  <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"><abbr class="badge">Cereb. Cortex</abbr></div> -->

        <!-- Entry bib key -->
        <div id="Song2021" class="col-sm-11">
        <!-- Title -->
        <div class="title">Maximum Entropy Principle Underlies Wiring Length Distribution in Brain Networks</div>
        <!-- Author -->
        <div class="author">
        

        Song, Yuru, , <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>, and <a href="https://ins.sjtu.edu.cn/people/songtingli/" rel="external nofollow noopener" target="_blank">Songting Li</a>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Cereb. Cortex</em>, 31(10), 4628–4641 2021
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://academic.oup.com/cercor/article/31/10/4628/6276649" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
            <a href="/assets/pdf/Song_Zhou_Li_CerebralCortex_2021.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1093/cercor/bhab110"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1093/cercor/bhab110" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>A brain network comprises a substantial amount of short-range connections with an admixture of long-range connections. The portion of long-range connections in brain networks is observed to be quantitatively dissimilar across species. It is hypothesized that the length of connections is constrained by the spatial embedding of brain networks, yet fundamental principles that underlie the wiring length distribution remain unclear. By quantifying the structural diversity of a brain network using Shannon’s entropy, here we show that the wiring length distribution across multiple species—including Drosophila, mouse, macaque, human, and C. elegans—follows the maximum entropy principle (MAP) under the constraints of limited wiring material and the spatial locations of brain areas or neurons. In addition, by considering stochastic axonal growth, we propose a network formation process capable of reproducing wiring length distributions of the 5 species, thereby implementing MAP in a biologically plausible manner. We further develop a generative model incorporating MAP, and show that, for the 5 species, the generated network exhibits high similarity to the real network. Our work indicates that the brain connectivity evolves to be structurally diversified by maximizing entropy to support efficient interareal communication, providing a potential organizational principle of brain networks.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Song2021</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Song, Yuru and Zhou, Douglas and Li, Songting}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1093/cercor/bhab110}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1047-3211}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Cereb. Cortex}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">aug</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{10}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{4628--4641}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Maximum Entropy Principle Underlies Wiring Length Distribution in Brain Networks}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{31}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="VanViegen2021" class="col-sm-11">
        <!-- Title -->
        <div class="title">Neuromatch Academy: Teaching Computational Neuroscience with Global Accessibility</div>
        <!-- Author -->
        <div class="author">
        

        Viegen, Tara, , Akrami, Athena, , 
                Bonnen, Kathryn, and
          <span class="more-authors" title="click to view 32 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '32 more authors' ? 'Eric DeWitt, Alexandre Hyafil, Helena Ledmyr, Grace W. Lindsay, Patrick Mineault, John D. Murray, Xaq Pitkow, Aina Puce, Madineh Sedigh-Sarvestani, Carsen Stringer, Titipat Achakulvisut, Elnaz Alikarami, Melvin Selim Atay, Eleanor Batty, Jeffrey C. Erlich, Byron V. Galbraith, Yueqi Guo, Ashley L. Juavinett, Matthew R. Krause, Songting Li, Marius Pachitariu, Elizabeth Straley, Davide Valeriani, Emma Vaughan, Maryam Vaziri-Pashkam, Michael L. Waskom, Gunnar Blohm, Konrad Kording, Paul Schrater, Brad Wyble, Sean Escola, Megan A.K. Peters' : '32 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">32 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Trends Cogn. Sci.</em>, 25(7), 535–538 2021
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://linkinghub.elsevier.com/retrieve/pii/S1364661321000954" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-arxiv-id="2012.08973"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1016/j.tics.2021.03.018" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Neuromatch Academy (NMA) designed and ran a fully online 3-week Computational Neuroscience Summer School for 1757 students with 191 teaching assistants (TAs) working in virtual inverted (or flipped) classrooms and on small group projects. Fourteen languages, active community management, and low cost allowed for an unprecedented level of inclusivity and universal accessibility.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">VanViegen2021</span><span class="p">,</span>
  <span class="na">archiveprefix</span> <span class="p">=</span> <span class="s">{arXiv}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{van Viegen, Tara and Akrami, Athena and Bonnen, Kathryn and DeWitt, Eric and Hyafil, Alexandre and Ledmyr, Helena and Lindsay, Grace W. and Mineault, Patrick and Murray, John D. and Pitkow, Xaq and Puce, Aina and Sedigh-Sarvestani, Madineh and Stringer, Carsen and Achakulvisut, Titipat and Alikarami, Elnaz and Atay, Melvin Selim and Batty, Eleanor and Erlich, Jeffrey C. and Galbraith, Byron V. and Guo, Yueqi and Juavinett, Ashley L. and Krause, Matthew R. and Li, Songting and Pachitariu, Marius and Straley, Elizabeth and Valeriani, Davide and Vaughan, Emma and Vaziri-Pashkam, Maryam and Waskom, Michael L. and Blohm, Gunnar and Kording, Konrad and Schrater, Paul and Wyble, Brad and Escola, Sean and Peters, Megan A.K.}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.tics.2021.03.018}</span><span class="p">,</span>
  <span class="na">eprint</span> <span class="p">=</span> <span class="s">{2012.08973}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{13646613}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Trends Cogn. Sci.}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Python coding,accessibility,computational neuroscience,data science,education,inclusivity}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jul</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{7}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{535--538}</span><span class="p">,</span>
  <span class="na">pmid</span> <span class="p">=</span> <span class="s">{33994097}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Neuromatch Academy: Teaching Computational Neuroscience with Global Accessibility}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{25}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Pyzza2021" class="col-sm-11">
        <!-- Title -->
        <div class="title">Network mechanism for insect olfaction</div>
        <!-- Author -->
        <div class="author">
        

        Pyzza, Pamela B., , Newhall, Katherine A., , 
                Kovačič, Gregor, and
          <span class="more-authors" title="click to view 2 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '2 more authors' ? 'Douglas Zhou, David Cai' : '2 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">2 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Cogn. Neurodyn.</em>, 15(1), 103–129 2021
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="http://link.springer.com/10.1007/s11571-020-09640-3" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-arxiv-id="1908.11865"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1007/s11571-020-09640-3" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Early olfactory pathway responses to the presentation of an odor exhibit remarkably similar dynamical behavior across phyla from insects to mammals, and frequently involve transitions among quiescence, collective network oscillations, and asynchronous firing. We hypothesize that the time scales of fast excitation and fast and slow inhibition present in these networks may be the essential element underlying this similar behavior, and design an idealized, conductance-based integrate-and-fire model to verify this hypothesis via numerical simulations. To better understand the mathematical structure underlying the common dynamical behavior across species, we derive a firing-rate model and use it to extract a slow passage through a saddle-node-on-an-invariant-circle bifurcation structure. We expect this bifurcation structure to provide new insights into the understanding of the dynamical behavior of neuronal assemblies and that a similar structure can be found in other sensory systems.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Pyzza2021</span><span class="p">,</span>
  <span class="na">archiveprefix</span> <span class="p">=</span> <span class="s">{arXiv}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Pyzza, Pamela B. and Newhall, Katherine A. and Kova{\v{c}}i{\v{c}}, Gregor and Zhou, Douglas and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/s11571-020-09640-3}</span><span class="p">,</span>
  <span class="na">eprint</span> <span class="p">=</span> <span class="s">{1908.11865}</span><span class="p">,</span>
  <span class="na">isbn</span> <span class="p">=</span> <span class="s">{0123456789}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1871-4080}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Cogn. Neurodyn.}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Antennal lobe,Firing-rate model,Gamma-band oscillations,Insect olfaction,Integrate-and-fire model,Saddle-node-on-an-invariant-circle bifurcation,Slow firing-rate patterns,Temporal binding}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">feb</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{103--129}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Network mechanism for insect olfaction}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{15}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
</ol>

  <h2 class="year">2020</h2>
  <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Crodelle2020" class="col-sm-11">
        <!-- Title -->
        <div class="title">A computational investigation of electrotonic coupling between pyramidal cells in the cortex</div>
        <!-- Author -->
        <div class="author">
        

        Crodelle, Jennifer, , <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>, 
                Kovačič, Gregor, and
          <span class="more-authors" title="click to view 1 more author" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '1 more author' ? 'David Cai' : '1 more author';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">1 more author</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>J. Comput. Neurosci.</em>, 48(4), 387–407 2020
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://link.springer.com/10.1007/s10827-020-00762-5" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1007/s10827-020-00762-5"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1007/s10827-020-00762-5" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>The existence of electrical communication among pyramidal cells (PCs) in the adult cortex has been debated by neuroscientists for several decades. Gap junctions (GJs) among cortical interneurons have been well documented experimentally and their functional roles have been proposed by both computational neuroscientists and experimentalists alike. Experimental evidence for similar junctions among pyramidal cells in the cortex, however, has remained elusive due to the apparent rarity of these couplings among neurons. In this work, we develop a neuronal network model that includes observed probabilities and strengths of electrotonic coupling between PCs and gap-junction coupling among interneurons, in addition to realistic synaptic connectivity among both populations. We use this network model to investigate the effect of electrotonic coupling between PCs on network behavior with the goal of theoretically addressing this controversy of existence and purpose of electrotonically coupled PCs in the cortex.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Crodelle2020</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Crodelle, Jennifer and Zhou, Douglas and Kova{\v{c}}i{\v{c}}, Gregor and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/s10827-020-00762-5}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0929-5313}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{J. Comput. Neurosci.}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Gap junctions,Hodgkin-Huxley model,Neuronal networks,Pyramidal cells}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">nov</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{4}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{387--407}</span><span class="p">,</span>
  <span class="na">pmid</span> <span class="p">=</span> <span class="s">{32892300}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{A computational investigation of electrotonic coupling between pyramidal cells in the cortex}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{48}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Wang2020" class="col-sm-11">
        <!-- Title -->
        <div class="title">Computational neuroscience: a frontier of the 21st century</div>
        <!-- Author -->
        <div class="author">
        

        Wang, Xiao-Jing, , Hu, Hailan, , 
                Huang, Chengcheng, and
          <span class="more-authors" title="click to view 11 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '11 more authors' ? 'Henry Kennedy, Chengyu Tony Li, Nikos Logothetis, Zhong-Lin Lu, Qingming Luo, Mu-ming Poo, Doris Tsao, Si Wu, Zhaohui Wu, Xu Zhang, Douglas Zhou' : '11 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">11 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Natl. Sci. Rev.</em>, 7(9), 1414–1418 2020
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://academic.oup.com/nsr/article/7/9/1414/5854209" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1093/nsr/nwaa123"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1093/nsr/nwaa123" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Wang2020</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Wang, Xiao-Jing and Hu, Hailan and Huang, Chengcheng and Kennedy, Henry and Li, Chengyu Tony and Logothetis, Nikos and Lu, Zhong-Lin and Luo, Qingming and Poo, Mu-ming and Tsao, Doris and Wu, Si and Wu, Zhaohui and Zhang, Xu and Zhou, Douglas}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1093/nsr/nwaa123}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2095-5138}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Natl. Sci. Rev.}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">sep</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{9}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1414--1418}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Computational neuroscience: a frontier of the 21st century}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{7}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="tian2020exponential" class="col-sm-11">
        <!-- Title -->
        <div class="title">Exponential Time Differencing Algorithm for Pulse-Coupled Hodgkin-Huxley Neural Networks</div>
        <!-- Author -->
        <div class="author">
        

        <a href="https://lcns-sjtu.github.io/people/zhongqi_tian/">Zhong-qi Kyle Tian</a>, and <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Front. Comput. Neurosci.</em>, 14, 40 2020
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://www.frontiersin.org/article/10.3389/fncom.2020.00040/full" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.3389/fncom.2020.00040"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.3389/fncom.2020.00040" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>The exponential time differencing (ETD) method allows using a large time step to efficiently evolve stiff systems such as Hodgkin-Huxley (HH) neural networks. For pulse-coupled HH networks, the synaptic spike times cannot be predetermined and are convoluted with neuron’s trajectory itself. This presents a challenging issue for the design of an efficient numerical simulation algorithm. The stiffness in the HH equations are quite different, for example, between the spike and non-spike regions. Here, we design a second-order adaptive exponential time differencing algorithm (AETD2) for the numerical evolution of HH neural networks. Compared with the regular second-order Runge-Kutta method (RK2), our AETD2 method can use time steps one order of magnitude larger and improve computational efficiency more than ten times while excellently capturing accurate traces of membrane potentials of HH neurons. This high accuracy and efficiency can be robustly obtained and do not depend on the dynamical regimes, connectivity structure or the network size.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">tian2020exponential</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Tian, Zhong-qi Kyle and Zhou, Douglas}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.3389/fncom.2020.00040}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1662-5188}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Front. Comput. Neurosci.}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Hodgkin-Huxley,efficiency,exponential time differencing method,pulse-coupled,second-order}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">may</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{40}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Frontiers}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Exponential Time Differencing Algorithm for Pulse-Coupled Hodgkin-Huxley Neural Networks}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{14}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Cheng2020" class="col-sm-11">
        <!-- Title -->
        <div class="title">The extended Granger causality analysis for Hodgkin-Huxley neuronal models</div>
        <!-- Author -->
        <div class="author">
        

        Cheng, Hong, , Cai, David, , and <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Chaos</em>, 30(10), 103102 2020
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1063/5.0006349"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1063/5.0006349" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>How to extract directions of information flow in dynamical systems based on empirical data remains a key challenge. The Granger causality (GC) analysis has been identified as a powerful method to achieve this capability. However, the framework of the GC theory requires that the dynamics of the investigated system can be statistically linearized; i.e., the dynamics can be effectively modeled by linear regressive processes. Under such conditions, the causal connectivity can be directly mapped to the structural connectivity that mediates physical interactions within the system. However, for nonlinear dynamical systems such as the Hodgkin-Huxley (HH) neuronal circuit, the validity of the GC analysis has yet been addressed; namely, whether the constructed causal connectivity is still identical to the synaptic connectivity between neurons remains unknown. In this work, we apply the nonlinear extension of the GC analysis, i.e., the extended GC analysis, to the voltage time series obtained by evolving the HH neuronal network. In addition, we add a certain amount of measurement or observational noise to the time series to take into account the realistic situation in data acquisition in the experiment. Our numerical results indicate that the causal connectivity obtained through the extended GC analysis is consistent with the underlying synaptic connectivity of the system. This consistency is also insensitive to dynamical regimes, e.g., a chaotic or non-chaotic regime. Since the extended GC analysis could in principle be applied to any nonlinear dynamical system as long as its attractor is low dimensional, our results may potentially be extended to the GC analysis in other settings.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Cheng2020</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Cheng, Hong and Cai, David and Zhou, Douglas}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1063/5.0006349}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{10897682}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Chaos}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{103102}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{10}</span><span class="p">,</span>
  <span class="na">pmid</span> <span class="p">=</span> <span class="s">{33138445}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{The extended Granger causality analysis for Hodgkin-Huxley neuronal models}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{30}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Tian2020" class="col-sm-11">
        <!-- Title -->
        <div class="title">A Combined Offline–Online Algorithm for Hodgkin–Huxley Neural Networks</div>
        <!-- Author -->
        <div class="author">
        

        Tian, Zhong qi Kyle, , Crodelle, Jennifer, , and <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>J. Sci. Comput.</em>, 84(1), 103102 2020
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://doi.org/10.1007/s10915-020-01261-6" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1007/s10915-020-01261-6"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1007/s10915-020-01261-6" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Spiking neural networks are widely applied to simulate cortical dynamics in the brain and are regarded as the next generation of machine learning. The classical Hodgkin–Huxley (HH) neuron is the foundation of all spiking neural models. In numerical simulation, however, the stiffness of the nonlinear HH equations during an action potential (a spike) period prohibits the use of large time steps for numerical integration. Outside of this stiff period, the HH equations can be efficiently simulated with a relatively large time step. In this work, we present an efficient and accurate offline–online combined method that stops evolving the HH equations during an action potential period, uses a pre-computed (offline) high-resolution data set to determine the voltage value during the spike, and restarts the time evolution of the HH equations after the stiff period using reset values interpolated from the offline data set. Our method allows for time steps an order of magnitude larger than those used in the standard Runge–Kutta (RK) method, while accurately capturing dynamical properties of HH neurons. In addition, this offline–online method robustly achieves a maximum of a tenfold decrease in computation time as compared to RK methods, a result that is independent of network size.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Tian2020</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Tian, Zhong qi Kyle and Crodelle, Jennifer and Zhou, Douglas}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/s10915-020-01261-6}</span><span class="p">,</span>
  <span class="na">isbn</span> <span class="p">=</span> <span class="s">{0123456789}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{15737691}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{J. Sci. Comput.}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Fast algorithm,Hodgkin–Huxley,Numerical simulation,Offline–online method}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Springer US}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{A Combined Offline–Online Algorithm for Hodgkin–Huxley Neural Networks}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{84}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Xu2020" class="col-sm-11">
        <!-- Title -->
        <div class="title">Neural networks of different species, brain areas and states can be characterized by the probability polling state</div>
        <!-- Author -->
        <div class="author">
        

        Xu, Zhi Qin John, , Gu, Xiaowei, , 
                Li, Chengyu, and
          <span class="more-authors" title="click to view 3 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '3 more authors' ? 'David Cai, Douglas Zhou, David W. McLaughlin' : '3 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">3 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Eur. J. Neurosci.</em>, 52(7), 3790–3802 2020
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1111/ejn.14860"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1111/ejn.14860" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Cortical networks are complex systems of a great many interconnected neurons that operate from collective dynamical states. To understand how cortical neural networks function, it is important to identify their common dynamical operating states from the probabilistic viewpoint. Probabilistic characteristics of these operating states often underlie network functions. Here, using multi-electrode data from three separate experiments, we identify and characterize a cortical operating state (the “probability polling” or “p-polling” state), common across mouse and monkey with different behaviors. If the interaction among neurons is weak, the p-polling state provides a quantitative understanding of how the high dimensional probability distribution of firing patterns can be obtained by the low-order maximum entropy formulation, effectively utilizing a low dimensional stimulus-coding structure. These results show evidence for generality of the p-polling state and in certain situations its advantage of providing a mathematical validation for the low-order maximum entropy principle as a coding strategy.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Xu2020</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Xu, Zhi Qin John and Gu, Xiaowei and Li, Chengyu and Cai, David and Zhou, Douglas and McLaughlin, David W.}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1111/ejn.14860}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{14609568}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Eur. J. Neurosci.}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{coding,cortical state,maximum entropy,network,neuroscience}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{7}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{3790--3802}</span><span class="p">,</span>
  <span class="na">pmid</span> <span class="p">=</span> <span class="s">{32533744}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Neural networks of different species, brain areas and states can be characterized by the probability polling state}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{52}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
</ol>

  <h2 class="year">2019</h2>
  <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Barranca2019a" class="col-sm-11">
        <!-- Title -->
        <div class="title">Compressive Sensing Inference of Neuronal Network Connectivity in Balanced Neuronal Dynamics</div>
        <!-- Author -->
        <div class="author">
        

        Barranca, Victor J., , and <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Front. Neurosci.</em>, 13, 1–10 2019
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://www.frontiersin.org/article/10.3389/fnins.2019.01101/full" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.3389/fnins.2019.01101"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.3389/fnins.2019.01101" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Determining the structure of a network is of central importance to understanding its function in both neuroscience and applied mathematics. However, recovering the structural connectivity of neuronal networks remains a fundamental challenge both theoretically and experimentally. While neuronal networks operate in certain dynamical regimes, which may influence their connectivity reconstruction, there is widespread experimental evidence of a balanced neuronal operating state in which strong excitatory and inhibitory inputs are dynamically adjusted such that neuronal voltages primarily remain near resting potential. Utilizing the dynamics of model neurons in such a balanced regime in conjunction with the ubiquitous sparse connectivity structure of neuronal networks, we develop a compressive sensing theoretical framework for efficiently reconstructing network connections by measuring individual neuronal activity in response to a relatively small ensemble of random stimuli injected over a short time scale. By tuning the network dynamical regime, we determine that the highest fidelity reconstructions are achievable in the balanced state. We hypothesize the balanced dynamics observed in vivo may therefore be a result of evolutionary selection for optimal information encoding and expect the methodology developed to be generalizable for alternative model networks as well as experimental paradigms.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Barranca2019a</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Barranca, Victor J. and Zhou, Douglas}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.3389/fnins.2019.01101}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1662-453X}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Front. Neurosci.}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Balanced networks,Connectivity reconstruction,Network dynamics,Neuronal networks,Signal processing}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">oct</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1--10}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Compressive Sensing Inference of Neuronal Network Connectivity in Balanced Neuronal Dynamics}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{13}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2019}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Gu2019a" class="col-sm-11">
        <!-- Title -->
        <div class="title">Emergence of spatially periodic diffusive waves in small-world neuronal networks</div>
        <!-- Author -->
        <div class="author">
        

        Gu, Qinglong L., , Xiao, Yanyang, , <a href="https://ins.sjtu.edu.cn/people/songtingli/" rel="external nofollow noopener" target="_blank">Songting Li</a>, and
          <span class="more-authors" title="click to view 1 more author" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '1 more author' ? 'Douglas Zhou' : '1 more author';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">1 more author</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Phys. Rev. E</em>, 100(4), 042401 2019
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://link.aps.org/doi/10.1103/PhysRevE.100.042401" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1103/PhysRevE.100.042401"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1103/PhysRevE.100.042401" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>It has been observed in experiment that the anatomical structure of neuronal networks in the brain possesses the feature of small-world networks. Yet how the small-world structure affects network dynamics remains to be fully clarified. Here we study the dynamics of a class of small-world networks consisting of pulse-coupled integrate-and-fire (I&amp;F) neurons. Under stochastic Poisson drive, we find that the activity of the entire network resembles diffusive waves. To understand its underlying mechanism, we analyze the simplified regular-lattice network consisting of firing-rate-based neurons as an approximation to the original I&amp;F small-world network. We demonstrate both analytically and numerically that, with strongly coupled connections, in the absence of noise, the activity of the firing-rate-based regular-lattice network spatially forms a static grating pattern that corresponds to the spatial distribution of the firing rate observed in the I&amp;F small-world neuronal network. We further show that the spatial grating pattern with different phases comprise the continuous attractor of both the I&amp;F small-world and firing-rate-based regular-lattice network dynamics. In the presence of input noise, the activity of both networks is perturbed along the continuous attractor, which gives rise to the diffusive waves. Our numerical simulations and theoretical analysis may potentially provide insights into the understanding of the generation of wave patterns observed in cortical networks.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Gu2019a</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Gu, Qinglong L. and Xiao, Yanyang and Li, Songting and Zhou, Douglas}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1103/PhysRevE.100.042401}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2470-0045}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Phys. Rev. E}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">oct</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{4}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{042401}</span><span class="p">,</span>
  <span class="na">pmid</span> <span class="p">=</span> <span class="s">{31770933}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{American Physical Society}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Emergence of spatially periodic diffusive waves in small-world neuronal networks}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{100}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2019}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"><abbr class="badge">PNAS</abbr></div> -->

        <!-- Entry bib key -->
        <div id="Li2019" class="col-sm-11">
        <!-- Title -->
        <div class="title">Dendritic computations captured by an effective point neuron model</div>
        <!-- Author -->
        <div class="author">
        

        <a href="https://ins.sjtu.edu.cn/people/songtingli/" rel="external nofollow noopener" target="_blank">Songting Li</a>, Liu, Nan, , 
                Zhang, Xiaohui, and
          <span class="more-authors" title="click to view 3 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '3 more authors' ? 'David W. McLaughlin, Douglas Zhou, David Cai' : '3 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">3 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Proc. Natl. Acad. Sci.</em>, 116(30), 15244–15252 2019
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.1904463116" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1073/pnas.1904463116"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1073/pnas.1904463116" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Complex dendrites in general present formidable challenges to understanding neuronal information processing. To circumvent the difficulty, a prevalent viewpoint simplifies the neuronal morphology as a point representing the soma, and the excitatory and inhibitory synaptic currents originated from the dendrites are treated as linearly summed at the soma. Despite its extensive applications, the validity of the synaptic current description remains unclear, and the existing point neuron framework fails to characterize the spatiotemporal aspects of dendritic integration supporting specific computations. Using electrophysiological experiments, realistic neuronal simulations, and theoretical analyses, we demonstrate that the traditional assumption of linear summation of synaptic currents is oversimplified and underestimates the inhibition effect. We then derive a form of synaptic integration current within the point neuron framework to capture dendritic effects. In the derived form, the interaction between each pair of synaptic inputs on the dendrites can be reliably parameterized by a single coefficient, suggesting the inherent low-dimensional structure of dendritic integration. We further generalize the form of synaptic integration current to capture the spatiotemporal interactions among multiple synaptic inputs and show that a point neuron model with the synaptic integration current incorporated possesses the computational ability of a spatial neuron with dendrites, including direction selectivity, coincidence detection, logical operation, and a bilinear dendritic integration rule discovered in experiment. Our work amends the modeling of synaptic inputs and improves the computational power of a modeling neuron within the point neuron framework.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Li2019</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Li, Songting and Liu, Nan and Zhang, Xiaohui and McLaughlin, David W. and Zhou, Douglas and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1073/pnas.1904463116}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0027-8424}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Proc. Natl. Acad. Sci.}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jul</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{30}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{15244--15252}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Dendritic computations captured by an effective point neuron model}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{116}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2019}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Crodelle2019" class="col-sm-11">
        <!-- Title -->
        <div class="title">A Role for Electrotonic Coupling Between Cortical Pyramidal Cells</div>
        <!-- Author -->
        <div class="author">
        

        Crodelle, Jennifer, , <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>, 
                Kovačič, Gregor, and
          <span class="more-authors" title="click to view 1 more author" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '1 more author' ? 'David Cai' : '1 more author';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">1 more author</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Front. Comput. Neurosci.</em>, 13, 1–13 2019
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://www.frontiersin.org/article/10.3389/fncom.2019.00033/full" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.3389/fncom.2019.00033"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.3389/fncom.2019.00033" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Many brain regions communicate information through synchronized network activity. Electrical coupling among the dendrites of interneurons in the cortex has been implicated in forming and sustaining such activity in the cortex. Evidence for the existence of electrical coupling among cortical pyramidal cells, however, has been largely absent. A recent experimental study measured properties of electrical connections between pyramidal cells in the cortex deemed “electrotonic couplings.” These junctions were seen to occur pair-wise, sparsely, and often coexist with electrically-coupled interneurons. Here, we construct a network model to investigate possible roles for these rare, electrotonically-coupled pyramidal-cell pairs. Through simulations, we show that electrical coupling among pyramidal-cell pairs significantly enhances coincidence-detection capabilities and increases network spike-timing precision. Further, a network containing multiple pairs exhibits large variability in its firing pattern, possessing a rich coding structure.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Crodelle2019</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Crodelle, Jennifer and Zhou, Douglas and Kova{\v{c}}i{\v{c}}, Gregor and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.3389/fncom.2019.00033}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1662-5188}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Front. Comput. Neurosci.}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Coincidence detection,Electrotonic coupling,Information processing,Pyramidal cells,Synchrony}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">may</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1--13}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{A Role for Electrotonic Coupling Between Cortical Pyramidal Cells}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{13}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2019}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Li2019determination" class="col-sm-11">
        <!-- Title -->
        <div class="title">Determination of effective synaptic conductances using somatic voltage clamp</div>
        <!-- Author -->
        <div class="author">
        

        <a href="https://ins.sjtu.edu.cn/people/songtingli/" rel="external nofollow noopener" target="_blank">Songting Li</a>, Liu, Nan, , 
                Yao, Li, and
          <span class="more-authors" title="click to view 3 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '3 more authors' ? 'Xiaohui Zhang, Douglas Zhou, David Cai' : '3 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">3 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>PLOS Comput. Biol.</em>, 15(3), 1-28 2019
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://dx.plos.org/10.1371/journal.pcbi.1006871" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-arxiv-id="1710.05098"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pcbi.1006871" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>The interplay between excitatory and inhibitory neurons imparts rich functions of the brain. To understand the synaptic mechanisms underlying neuronal computations, a fundamental approach is to study the dynamics of excitatory and inhibitory synaptic inputs of each neuron. The traditional method of determining input conductance, which has been applied for decades, employs the synaptic current-voltage (I-V) relation obtained via voltage clamp. Due to the space clamp effect, the measured conductance is different from the local conductance on the dendrites. Therefore, the interpretation of the measured conductance remains to be clarified. Using theoretical analysis, electrophysiological experiments, and realistic neuron simulations, here we demonstrate that there does not exist a transform between the local conductance and the conductance measured by the traditional method, due to the neglect of a nonlinear interaction between the clamp current and the synaptic current in the traditional method. Consequently, the conductance determined by the traditional method may not correlate with the local conductance on the dendrites, and its value could be unphysically negative as observed in experiment. To circumvent the challenge of the space clamp effect and elucidate synaptic impact on neuronal information processing, we propose the concept of effective conductance which is proportional to the local conductance on the dendrite and reflects directly the functional influence of synaptic inputs on somatic membrane potential dynamics, and we further develop a framework to determine the effective conductance accurately. Our work suggests re-examination of previous studies involving conductance measurement and provides a reliable approach to assess synaptic influence on neuronal computation.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Li2019determination</span><span class="p">,</span>
  <span class="na">archiveprefix</span> <span class="p">=</span> <span class="s">{arXiv}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Li, Songting and Liu, Nan and Yao, Li and Zhang, Xiaohui and Zhou, Douglas and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1371/journal.pcbi.1006871}</span><span class="p">,</span>
  <span class="na">editor</span> <span class="p">=</span> <span class="s">{Hennig, Matthias Helge}</span><span class="p">,</span>
  <span class="na">eprint</span> <span class="p">=</span> <span class="s">{1710.05098}</span><span class="p">,</span>
  <span class="na">isbn</span> <span class="p">=</span> <span class="s">{1111111111}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1553-7358}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{PLOS Comput. Biol.}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">mar</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{3}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1-28}</span><span class="p">,</span>
  <span class="na">pmid</span> <span class="p">=</span> <span class="s">{30835719}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Determination of effective synaptic conductances using somatic voltage clamp}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{15}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2019}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Xu2019a" class="col-sm-11">
        <!-- Title -->
        <div class="title">Maximum entropy principle analysis in network systems with short-time recordings</div>
        <!-- Author -->
        <div class="author">
        

        Xu, Zhi-Qin John, , Crodelle, Jennifer, , <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>, and
          <span class="more-authors" title="click to view 1 more author" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '1 more author' ? 'David Cai' : '1 more author';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">1 more author</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Phys. Rev. E</em>, 99(2), 022409 2019
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://link.aps.org/doi/10.1103/PhysRevE.99.022409" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-arxiv-id="1808.10506"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1103/PhysRevE.99.022409" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>In many realistic systems, maximum entropy principle (MEP) analysis provides an effective characterization of the probability distribution of network states. However, to implement the MEP analysis, a sufficiently long-time data recording in general is often required, e.g., hours of spiking recordings of neurons in neuronal networks. The issue of whether the MEP analysis can be successfully applied to network systems with data from short-time recordings has yet to be fully addressed. In this work, we investigate relationships underlying the probability distributions, moments, and effective interactions in the MEP analysis and then show that, with short-time recordings of network dynamics, the MEP analysis can be applied to reconstructing probability distributions of network states that is much more accurate than the one directly measured from the short-time recording. Using spike trains obtained from both Hodgkin-Huxley neuronal networks and electrophysiological experiments, we verify our results and demonstrate that MEP analysis provides a tool to investigate the neuronal population coding properties for short-time recordings.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Xu2019a</span><span class="p">,</span>
  <span class="na">archiveprefix</span> <span class="p">=</span> <span class="s">{arXiv}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Xu, Zhi-Qin John and Crodelle, Jennifer and Zhou, Douglas and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1103/PhysRevE.99.022409}</span><span class="p">,</span>
  <span class="na">eprint</span> <span class="p">=</span> <span class="s">{1808.10506}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2470-0045}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Phys. Rev. E}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">feb</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{2}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{022409}</span><span class="p">,</span>
  <span class="na">pmid</span> <span class="p">=</span> <span class="s">{30934291}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{American Physical Society}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Maximum entropy principle analysis in network systems with short-time recordings}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{99}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2019}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Gu2019" class="col-sm-11">
        <!-- Title -->
        <div class="title">Balanced Active Core in Heterogeneous Neuronal Networks</div>
        <!-- Author -->
        <div class="author">
        

        Gu, Qing-long L., , <a href="https://ins.sjtu.edu.cn/people/songtingli/" rel="external nofollow noopener" target="_blank">Songting Li</a>, 
                Dai, Wei P., and
          <span class="more-authors" title="click to view 2 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '2 more authors' ? 'Douglas Zhou, David Cai' : '2 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">2 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Front. Comput. Neurosci.</em>, 12, 022409 2019
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://www.frontiersin.org/article/10.3389/fncom.2018.00109/full" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.3389/fncom.2018.00109"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.3389/fncom.2018.00109" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>It is hypothesized that cortical neuronal circuits operate in a global balanced state, i.e., the majority of neurons fire irregularly by receiving balanced inputs of excitation and inhibition. Meanwhile, it has been observed in experiments that sensory information is often sparsely encoded by only a small set of firing neurons, while neurons in the rest of the network are silent. The phenomenon of sparse coding challenges the hypothesis of a global balanced state in the brain. To reconcile this, here we address the issue of whether a balanced state can exist in a small number of firing neurons by taking account of the heterogeneity of network structure such as scale-free and small-world networks. We propose necessary conditions and show that, under these conditions, for sparsely but strongly connected heterogeneous networks with various types of single-neuron dynamics, despite the fact that the whole network receives external inputs, there is a small active subnetwork (active core) inherently embedded within it. The neurons in this active core have relatively high firing rates while the neurons in the rest of the network are quiescent. Surprisingly, although the whole network is heterogeneous and unbalanced, the active core possesses a balanced state and its connectivity structure is close to a homogeneous Erdös-Rényi network. The dynamics of the active core can be well-predicted using the Fokker-Planck equation. Our results suggest that the balanced state may be maintained by a small group of spiking neurons embedded in a large heterogeneous network in the brain. The existence of the small active core reconciles the balanced state and the sparse coding, and also provides a potential dynamical scenario underlying sparse coding in neuronal networks.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Gu2019</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Gu, Qing-long L. and Li, Songting and Dai, Wei P. and Zhou, Douglas and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.3389/fncom.2018.00109}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1662-5188}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Front. Comput. Neurosci.}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Active core,Balanced state,Fokker-Planck equation,Heterogeneous,Homogeneous,Sparse coding}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jan</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Balanced Active Core in Heterogeneous Neuronal Networks}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{12}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2019}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Xu2019" class="col-sm-11">
        <!-- Title -->
        <div class="title">Dynamical and Coupling Structure of Pulse-Coupled Networks in Maximum Entropy Analysis</div>
        <!-- Author -->
        <div class="author">
        

        Xu, Zhi-Qin, , <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>, and  and 
                Cai, David</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Entropy</em>, 21(1), 76 2019
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="http://www.mdpi.com/1099-4300/21/1/76" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-arxiv-id="1808.04499"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.3390/e21010076" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Maximum entropy principle (MEP) analysis with few non-zero effective interactions successfully characterizes the distribution of dynamical states of pulse-coupled networks in many fields, e.g., in neuroscience. To better understand the underlying mechanism, we found a relation between the dynamical structure, i.e., effective interactions in MEP analysis, and the anatomical coupling structure of pulse-coupled networks and it helps to understand how a sparse coupling structure could lead to a sparse coding by effective interactions. This relation quantitatively displays how the dynamical structure is closely related to the anatomical coupling structure.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Xu2019</span><span class="p">,</span>
  <span class="na">archiveprefix</span> <span class="p">=</span> <span class="s">{arXiv}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Xu, Zhi-Qin and Zhou, Douglas and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.3390/e21010076}</span><span class="p">,</span>
  <span class="na">eprint</span> <span class="p">=</span> <span class="s">{1808.04499}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1099-4300}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Entropy}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Coupling structure,Maximum entropy,Neural network,Pulse-coupled network,Sparse coding}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jan</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{76}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Dynamical and Coupling Structure of Pulse-Coupled Networks in Maximum Entropy Analysis}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{21}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2019}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Dai2019" class="col-sm-11">
        <!-- Title -->
        <div class="title">Fast algorithms for simulation of neuronal dynamics based on the bilinear dendritic integration rule</div>
        <!-- Author -->
        <div class="author">
        

        Dai, Wei P., , <a href="https://ins.sjtu.edu.cn/people/songtingli/" rel="external nofollow noopener" target="_blank">Songting Li</a>, and <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Commun. Math. Sci.</em>, 17(5), 1313–1331 2019
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://www.intlpress.com/site/pub/pages/journals/items/cms/content/vols/0017/0005/a007/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.4310/CMS.2019.v17.n5.a7"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.4310/CMS.2019.v17.n5.a7" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>We aim to develop fast algorithms for neuronal simulations to capture the dynamics of a neuron with realistic dendritic morphology. To achieve this, we perform the asymptotic analysis on a cable neuron model with branched dendrites. Using the second-order asymptotic solutions, we derive a bilinear dendritic integration rule to characterize the voltage response at the soma when receiving multiple spatiotemporal synaptic inputs from dendrites, with a dependency on the voltage state of the neuron at input arrival times. Based on the derived bilinear rule, we finally propose two fast algorithms and demonstrate numerically that, in comparison with solving the original cable neuron model numerically, the algorithms can reduce the computational cost of simulation for neuronal dynamics enormously while retaining relatively high accuracy in terms of both sub-threshold dynamics and firing statistics.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Dai2019</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Dai, Wei P. and Li, Songting and Zhou, Douglas}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.4310/CMS.2019.v17.n5.a7}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{15396746}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Commun. Math. Sci.}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Asymptotic analysis,Bilinear rule,Cable equation,Dendrites,Dendritic integration}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{5}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1313--1331}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Fast algorithms for simulation of neuronal dynamics based on the bilinear dendritic integration rule}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{17}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2019}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Barranca2019" class="col-sm-11">
        <!-- Title -->
        <div class="title">The role of sparsity in inverse problems for networks with nonlinear dynamics</div>
        <!-- Author -->
        <div class="author">
        

        Barranca, Victor J., , Kovačič, Gregor, , and <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Commun. Math. Sci.</em>, 17(5), 1291–1311 2019
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://www.intlpress.com/site/pub/pages/journals/items/cms/content/vols/0017/0005/a006/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.4310/CMS.2019.v17.n5.a6"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.4310/CMS.2019.v17.n5.a6" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Sparsity is a fundamental characteristic of numerous biological, social, and technological networks. Network connectivity frequently demonstrates sparsity on multiple spatial scales and network inputs may also possess sparse representations in appropriate domains. In this work, we address the role of sparsity for solving inverse problems in networks with nonlinear and time-evolving dynamics. In the context of pulse-coupled integrate-and-fire networks, we demonstrate that nonlinear network dynamics imparts a compressive coding of both network connectivity and inputs provided they possess a sparse structure. This work thereby formulates an efficient sparsity-based framework for solving several classes of inverse problems in nonlinear network dynamics. Driving the network with a small ensemble of random inputs, we derive a mean-field set of underdetermined linear systems relating the network inputs to the corresponding activity of the nodes via the feed-forward connectivity matrix. In reconstructing the network connections, we utilize compressive sensing theory, which facilitates the recovery of sparse solutions to such underdetermined linear systems. Using the reconstructed connectivity matrix, we are capable of accurately recovering detailed network inputs, which may vary in time, distinct from the random input ensemble. This framework underlines the central role of sparsity in information transmission through network dynamics, providing new insight into the structure-function relationship for high-dimensional networks with nonlinear dynamics. Considering the reconstruction of structural connectivity in large networks is a significant and challenging problem in the study of complex networks, we hypothesize that analogous reconstruction methods taking advantage of sparsity may facilitate key advances in the field.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Barranca2019</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Barranca, Victor J. and Kova{\v{c}}i{\v{c}}, Gregor and Zhou, Douglas}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.4310/CMS.2019.v17.n5.a6}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{15396746}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Commun. Math. Sci.}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Compressive sensing,Inverse problems,Network re-construction,Neuronal networks,Nonlinear dynamics,Sparsity}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{5}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1291--1311}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{The role of sparsity in inverse problems for networks with nonlinear dynamics}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{17}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2019}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Xiao2019" class="col-sm-11">
        <!-- Title -->
        <div class="title">Representing conditional Granger causality by vector auto-regressive parameters</div>
        <!-- Author -->
        <div class="author">
        

        Xiao, Yanyang, , <a href="https://ins.sjtu.edu.cn/people/songtingli/" rel="external nofollow noopener" target="_blank">Songting Li</a>, and <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Commun. Math. Sci.</em>, 17(5), 1353–1386 2019
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://www.intlpress.com/site/pub/pages/journals/items/cms/content/vols/0017/0005/a009/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.4310/CMS.2019.v17.n5.a9"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.4310/CMS.2019.v17.n5.a9" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Granger Causality (GC) has been widely applied to various scientific fields to reveal causal relationships between dynamical variables. The mathematical framework of GC is based on the vector auto-regression (VAR) model, and the GC value from one variable to the other is defined as the logarithmic ratio of the variance of two prediction errors obtained by excluding and including the second variable in the VAR model respectively. Besides its definition, GC shall also be reflected in the regression parameters of the VAR model, e.g., larger regression coefficients indicate stronger causal interactions in general. Yet an explicit description of how the GC value depends on the VAR parameters for a general multi-variable case remains lacking. In this work, we aim to bridge this gap by expressing conditional GC using the VAR parameters, which provides an alternative interpretation of GC with novel intuition. The analysis developed in this work may also benefit the study of the VAR model in the future.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Xiao2019</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Xiao, Yanyang and Li, Songting and Zhou, Douglas}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.4310/CMS.2019.v17.n5.a9}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{15396746}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Commun. Math. Sci.}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Approximation,Granger Causality,VAR parameters}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{5}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1353--1386}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Representing conditional Granger causality by vector auto-regressive parameters}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{17}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2019}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
</ol>

  <h2 class="year">2018</h2>
  <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Dai2018" class="col-sm-11">
        <!-- Title -->
        <div class="title">Mechanisms underlying contrast-dependent orientation selectivity in mouse V1</div>
        <!-- Author -->
        <div class="author">
        

        Dai, Wei P., , <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>, 
                McLaughlin, David W., and
          <span class="more-authors" title="click to view 1 more author" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '1 more author' ? 'David Cai' : '1 more author';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">1 more author</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Proc. Natl. Acad. Sci.</em>, 115(45), 11619–11624 2018
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://pnas.org/doi/full/10.1073/pnas.1719044115" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1073/pnas.1719044115"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1073/pnas.1719044115" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Recently, sophisticated optogenetic tools for mouse have enabled many detailed studies of the neuronal circuits of its primary visual cortex (V1), providing much more specific information than is available for cat or monkey. Among various other differences, they show a striking contrast dependency in orientation selectivity in mouse V1 rather than the well-known contrast invariance for cat and monkey. Constrained by the existing experiment data, we develop a comprehensive large-scale model of an effective input layer of mouse V1 that successfully reproduces the contrast-dependent phenomena and many other response properties. The model helps to probe different mechanisms based on excitation–inhibition balance that underlie both contrast dependencies and invariance, and it provides implications for future studies on these circuits.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Dai2018</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Dai, Wei P. and Zhou, Douglas and McLaughlin, David W. and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1073/pnas.1719044115}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0027-8424}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Proc. Natl. Acad. Sci.}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Contrast dependence,Contrast invariance,Orientation selectivity}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">nov</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{45}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{11619--11624}</span><span class="p">,</span>
  <span class="na">pmid</span> <span class="p">=</span> <span class="s">{30337480}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Mechanisms underlying contrast-dependent orientation selectivity in mouse V1}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{115}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2018}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Gu2018" class="col-sm-11">
        <!-- Title -->
        <div class="title">The Dynamics of Balanced Spiking Neuronal Networks Under Poisson Drive Is Not Chaotic</div>
        <!-- Author -->
        <div class="author">
        

        Gu, Qing-long L., , <a href="https://lcns-sjtu.github.io/people/zhongqi_tian/">Zhong-qi K. Tian</a>, 
                Kovačič, Gregor, and
          <span class="more-authors" title="click to view 2 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '2 more authors' ? 'Douglas Zhou, David Cai' : '2 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">2 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Front. Comput. Neurosci.</em>, 12, 1–9 2018
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://www.frontiersin.org/article/10.3389/fncom.2018.00047/full" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.3389/fncom.2018.00047"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.3389/fncom.2018.00047" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Some previous studies have shown that chaotic dynamics in the balanced state, i.e., one with balanced excitatory and inhibitory inputs into cortical neurons, is the underlying mechanism for the irregularity of neural activity. In this work, we focus on networks of current-based integrate-and-fire neurons with delta-pulse coupling. While we show that the balanced state robustly persists in this system within a broad range of parameters, we mathematically prove that the largest Lyapunov exponent of this type of neuronal networks is negative. Therefore, the irregular firing activity can exist in the systemwithout the chaotic dynamics. That is the irregularity of balanced neuronal networks need not arise from chaos.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Gu2018</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Gu, Qing-long L. and Tian, Zhong-qi K. and Kova{\v{c}}i{\v{c}}, Gregor and Zhou, Douglas and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.3389/fncom.2018.00047}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1662-5188}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Front. Comput. Neurosci.}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Balanced state,Chaotic dynamics,Delta-pulse coupling,Irregular activity,Largest Lyapunov exponent}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jun</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1--9}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{The Dynamics of Balanced Spiking Neuronal Networks Under Poisson Drive Is Not Chaotic}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{12}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2018}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="li2018causal" class="col-sm-11">
        <!-- Title -->
        <div class="title">Causal inference in nonlinear systems: Granger causality versus time-delayed mutual information</div>
        <!-- Author -->
        <div class="author">
        

        <a href="https://ins.sjtu.edu.cn/people/songtingli/" rel="external nofollow noopener" target="_blank">Songting Li</a>, Xiao, Yanyang, , <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>, and
          <span class="more-authors" title="click to view 1 more author" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '1 more author' ? 'David Cai' : '1 more author';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">1 more author</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Phys. Rev. E</em>, 97(5), 052216 2018
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://link.aps.org/doi/10.1103/PhysRevE.97.052216" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1103/PhysRevE.97.052216"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1103/PhysRevE.97.052216" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>The Granger causality (GC) analysis has been extensively applied to infer causal interactions in dynamical systems arising from economy and finance, physics, bioinformatics, neuroscience, social science, and many other fields. In the presence of potential nonlinearity in these systems, the validity of the GC analysis in general is questionable. To illustrate this, here we first construct minimal nonlinear systems and show that the GC analysis fails to infer causal relations in these systems - it gives rise to all types of incorrect causal directions. In contrast, we show that the time-delayed mutual information (TDMI) analysis is able to successfully identify the direction of interactions underlying these nonlinear systems. We then apply both methods to neuroscience data collected from experiments and demonstrate that the TDMI analysis but not the GC analysis can identify the direction of interactions among neuronal signals. Our work exemplifies inference hazards in the GC analysis in nonlinear systems and suggests that the TDMI analysis can be an appropriate tool in such a case.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">li2018causal</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Li, Songting and Xiao, Yanyang and Zhou, Douglas and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1103/PhysRevE.97.052216}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2470-0045}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Phys. Rev. E}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">may</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{5}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{052216}</span><span class="p">,</span>
  <span class="na">pmid</span> <span class="p">=</span> <span class="s">{29906860}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{APS}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Causal inference in nonlinear systems: Granger causality versus time-delayed mutual information}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{97}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2018}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Min2018" class="col-sm-11">
        <!-- Title -->
        <div class="title">Effects of Firing Variability on Network Structures with Spike-Timing-Dependent Plasticity</div>
        <!-- Author -->
        <div class="author">
        

        Min, Bin, , <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>, and  and 
                Cai, David</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Front. Comput. Neurosci.</em>, 12, 052216 2018
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="http://journal.frontiersin.org/article/10.3389/fncom.2018.00001/full" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.3389/fncom.2018.00001"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.3389/fncom.2018.00001" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Synaptic plasticity is believed to be the biological substrate underlying learning and memory. One of the most widespread forms of synaptic plasticity, spike-timing-dependent plasticity (STDP), uses the spike timing information of presynaptic and postsynaptic neurons to induce synaptic potentiation or depression. An open question is how STDP organizes the connectivity patterns in neuronal circuits. Previous studies have placed much emphasis on the role of firing rate in shaping connectivity patterns. Here, we go beyond the firing rate description to develop a self-consistent linear response theory that incorporates the information of both firing rate and firing variability. By decomposing the pairwise spike correlation into one component associated with local direct connections and the other associated with indirect connections, we identify two distinct regimes regarding the network structures learned through STDP. In one regime, the contribution of the direct-connection correlations dominates over that of the indirect-connection correlations in the learning dynamics; this gives rise to a network structure consistent with the firing rate description. In the other regime, the contribution of the indirect-connection correlations dominates in the learning dynamics, leading to a network structure different from the firing rate description. We demonstrate that the heterogeneity of firing variability across neuronal populations induces a temporally asymmetric structure of indirect-connection correlations. This temporally asymmetric structure underlies the emergence of the second regime. Our study provides a new perspective that emphasizes the role of high-order statistics of spiking activity in the spike-correlation-sensitive learning dynamics.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Min2018</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Min, Bin and Zhou, Douglas and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.3389/fncom.2018.00001}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1662-5188}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Front. Comput. Neurosci.}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Correlation structure,Firing variability,Linear response theory,STDP,Synaptic plasticity}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jan</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Effects of Firing Variability on Network Structures with Spike-Timing-Dependent Plasticity}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{12}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2018}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
</ol>

  <h2 class="year">2017</h2>
  <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Zhang2017" class="col-sm-11">
        <!-- Title -->
        <div class="title">Spike-Triggered Regression for Synaptic Connectivity Reconstruction in Neuronal Networks</div>
        <!-- Author -->
        <div class="author">
        

        Zhang, Yaoyu, , Xiao, Yanyang, , <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>, and
          <span class="more-authors" title="click to view 1 more author" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '1 more author' ? 'David Cai' : '1 more author';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">1 more author</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Front. Comput. Neurosci.</em>, 11, 1–13 2017
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="http://journal.frontiersin.org/article/10.3389/fncom.2017.00101/full" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.3389/fncom.2017.00101"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.3389/fncom.2017.00101" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>How neurons are connected in the brain to perform computation is a key issue in neuroscience. Recently, the development of calcium imaging and multi-electrode array techniques have greatly enhanced our ability to measure the firing activities of neuronal populations at single cell level. Meanwhile, the intracellular recording technique is able to measure subthreshold voltage dynamics of a neuron. Our work addresses the issue of how to combine these measurements to reveal the underlying network structure. We propose the spike-triggered regression (STR) method, which employs both the voltage trace and firing activity of the neuronal population to reconstruct the underlying synaptic connectivity. Our numerical study of the conductance-based integrate-and-fire neuronal network shows that only short data of 20 ∼ 100 s is required for an accurate recovery of network topology as well as the corresponding coupling strength. Our method can yield an accurate reconstruction of a large neuronal network even in the case of dense connectivity and nearly synchronous dynamics, whichmany other network reconstruction methods cannot successfully handle. In addition, we point out that, for sparse networks, the STR method can infer coupling strength between each pair of neurons with high accuracy in the absence of the global information of all other neurons.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Zhang2017</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zhang, Yaoyu and Xiao, Yanyang and Zhou, Douglas and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.3389/fncom.2017.00101}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1662-5188}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Front. Comput. Neurosci.}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Coupling strength inference,Inference invariance,Network reconstruction,Neuronal dynamics,Spike-triggered regression}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">nov</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1--13}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Spike-Triggered Regression for Synaptic Connectivity Reconstruction in Neuronal Networks}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{11}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2017}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Li2017" class="col-sm-11">
        <!-- Title -->
        <div class="title">The characterization of hippocampal theta-driving neurons-A time-delayed mutual information approach</div>
        <!-- Author -->
        <div class="author">
        

        <a href="https://ins.sjtu.edu.cn/people/songtingli/" rel="external nofollow noopener" target="_blank">Songting Li</a>, Xu, Jiamin, , 
                Chen, Guifen, and
          <span class="more-authors" title="click to view 3 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '3 more authors' ? 'Longnian Lin, Douglas Zhou, David Cai' : '3 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">3 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Sci. Rep.</em>, 7(1), 1–12 2017
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="http://dx.doi.org/10.1038/s41598-017-05527-2" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1038/s41598-017-05527-2"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1038/s41598-017-05527-2" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Interneurons are important for computation in the brain, in particular, in the information processing involving the generation of theta oscillations in the hippocampus. Yet the functional role of interneurons in the theta generation remains to be elucidated. Here we use time-delayed mutual information to investigate information flow related to a special class of interneurons-theta-driving neurons in the hippocampal CA1 region of the mouse-to characterize the interactions between theta-driving neurons and theta oscillations. For freely behaving mice, our results show that information flows from the activity of theta-driving neurons to the theta wave, and the firing activity of theta-driving neurons shares a substantial amount of information with the theta wave regardless of behavioral states. Via realistic simulations of a CA1 pyramidal neuron, we further demonstrate that theta-driving neurons possess the characteristics of the cholecystokinin-expressing basket cells (CCK-BC). Our results suggest that it is important to take into account the role of CCK-BC in the generation and information processing of theta oscillations.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Li2017</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Li, Songting and Xu, Jiamin and Chen, Guifen and Lin, Longnian and Zhou, Douglas and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41598-017-05527-2}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{20452322}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Sci. Rep.}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1--12}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Springer US}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{The characterization of hippocampal theta-driving neurons-A time-delayed mutual information approach}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{7}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2017}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Xu2017" class="col-sm-11">
        <!-- Title -->
        <div class="title">A dynamical state underlying the second order maximum entropy principle in neuronal networks</div>
        <!-- Author -->
        <div class="author">
        

        Xu, Zhi-Qin John, , Bi, Guoqiang, , <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>, and
          <span class="more-authors" title="click to view 1 more author" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '1 more author' ? 'David Cai' : '1 more author';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">1 more author</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Commun. Math. Sci.</em>, 15(3), 665–692 2017
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="http://www.intlpress.com/site/pub/pages/journals/items/cms/content/vols/0015/0003/a005/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.4310/CMS.2017.v15.n3.a5"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.4310/CMS.2017.v15.n3.a5" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>The maximum entropy principle is widely used in diverse fields. We address the issue of why the second order maximum entropy model, by using only firing rates and second order correlations of neurons as constraints, can well capture the observed distribution of neuronal firing patterns in many neuronal networks, thus, conferring its great advantage in that the degree of complexity in the analysis of neuronal activity data reduces drastically from [inline-equation] to [inline-equation], where n is the number of neurons under consideration. We first derive an expression for the effective interactions of the nth order maximum entropy model using all orders of correlations of neurons as constraints and show that there exists a recursive relation among the effective interactions in the model. Then, via a perturbative analysis, we explore a possible dynamical state in which this recursive relation gives rise to the strengths of higher order interactions always smaller than the lower orders. Finally, we invoke this hierarchy of effective interactions to provide a possible mechanism underlying the success of the second order maximum entropy model and to predict whether such a model can successfully capture the observed distribution of neuronal firing patterns.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Xu2017</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Xu, Zhi-Qin John and Bi, Guoqiang and Zhou, Douglas and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.4310/CMS.2017.v15.n3.a5}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{15396746}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Commun. Math. Sci.}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{High dimensional data,Maximum entropy model,Mechanism,Neuronal networks}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{3}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{665--692}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{A dynamical state underlying the second order maximum entropy principle in neuronal networks}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{15}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2017}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
</ol>

  <h2 class="year">2016</h2>
  <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Barranca2016b" class="col-sm-11">
        <!-- Title -->
        <div class="title">Improved Compressive Sensing of Natural Scenes Using Localized Random Sampling</div>
        <!-- Author -->
        <div class="author">
        

        Barranca, Victor J., , Kovačič, Gregor, , <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>, and
          <span class="more-authors" title="click to view 1 more author" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '1 more author' ? 'David Cai' : '1 more author';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">1 more author</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Sci. Rep.</em>, 6(1), 31976 2016
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="http://dx.doi.org/10.1038/srep31976" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1038/srep31976"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1038/srep31976" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Compressive sensing (CS) theory demonstrates that by using uniformly-random sampling, rather than uniformly-spaced sampling, higher quality image reconstructions are often achievable. Considering that the structure of sampling protocols has such a profound impact on the quality of image reconstructions, we formulate a new sampling scheme motivated by physiological receptive field structure, localized random sampling, which yields significantly improved CS image reconstructions. For each set of localized image measurements, our sampling method first randomly selects an image pixel and then measures its nearby pixels with probability depending on their distance from the initially selected pixel. We compare the uniformly-random and localized random sampling methods over a large space of sampling parameters, and show that, for the optimal parameter choices, higher quality image reconstructions can be consistently obtained by using localized random sampling. In addition, we argue that the localized random CS optimal parameter choice is stable with respect to diverse natural images, and scales with the number of samples used for reconstruction. We expect that the localized random sampling protocol helps to explain the evolutionarily advantageous nature of receptive field structure in visual systems and suggests several future research areas in CS theory and its application to brain imaging.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Barranca2016b</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Barranca, Victor J. and Kova{\v{c}}i{\v{c}}, Gregor and Zhou, Douglas and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/srep31976}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2045-2322}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Sci. Rep.}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">aug</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{31976}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Nature Publishing Group}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Improved Compressive Sensing of Natural Scenes Using Localized Random Sampling}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{6}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2016}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Barranca2016" class="col-sm-11">
        <!-- Title -->
        <div class="title">Compressive sensing reconstruction of feed-forward connectivity in pulse-coupled nonlinear networks</div>
        <!-- Author -->
        <div class="author">
        

        Barranca, Victor J., , <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>, and  and 
                Cai, David</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Phys. Rev. E</em>, 93(6), 060201 2016
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://link.aps.org/doi/10.1103/PhysRevE.93.060201" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1103/PhysRevE.93.060201"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1103/PhysRevE.93.060201" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Utilizing the sparsity ubiquitous in real-world network connectivity, we develop a theoretical framework for efficiently reconstructing sparse feed-forward connections in a pulse-coupled nonlinear network through its output activities. Using only a small ensemble of random inputs, we solve this inverse problem through the compressive sensing theory based on a hidden linear structure intrinsic to the nonlinear network dynamics. The accuracy of the reconstruction is further verified by the fact that complex inputs can be well recovered using the reconstructed connectivity. We expect this Rapid Communication provides a new perspective for understanding the structure-function relationship as well as compressive sensing principle in nonlinear network dynamics.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Barranca2016</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Barranca, Victor J. and Zhou, Douglas and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1103/PhysRevE.93.060201}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2470-0045}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Phys. Rev. E}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jun</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{6}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{060201}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Compressive sensing reconstruction of feed-forward connectivity in pulse-coupled nonlinear networks}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{93}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2016}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Zhang2016" class="col-sm-11">
        <!-- Title -->
        <div class="title">Granger causality analysis with nonuniform sampling and its application to pulse-coupled nonlinear dynamics</div>
        <!-- Author -->
        <div class="author">
        

        Zhang, Yaoyu, , Xiao, Yanyang, , <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>, and
          <span class="more-authors" title="click to view 1 more author" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '1 more author' ? 'David Cai' : '1 more author';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">1 more author</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Phys. Rev. E</em>, 93(4), 042217 2016
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://link.aps.org/doi/10.1103/PhysRevE.93.042217" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1103/PhysRevE.93.042217"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1103/PhysRevE.93.042217" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>The Granger causality (GC) analysis is an effective approach to infer causal relations for time series. However, for data obtained by uniform sampling (i.e., with an equal sampling time interval), it is known that GC can yield unreliable causal inference due to aliasing if the sampling rate is not sufficiently high. To solve this unreliability issue, we consider the nonuniform sampling scheme as it can mitigate against aliasing. By developing an unbiased estimation of power spectral density of nonuniformly sampled time series, we establish a framework of spectrum-based nonparametric GC analysis. Applying this framework to a general class of pulse-coupled nonlinear networks and utilizing some particular spectral structure possessed by these nonlinear network data, we demonstrate that, for such nonlinear networks with nonuniformly sampled data, reliable GC inference can be achieved at a low nonuniform mean sampling rate at which the traditional uniform sampling GC may lead to spurious causal inference.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Zhang2016</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zhang, Yaoyu and Xiao, Yanyang and Zhou, Douglas and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1103/PhysRevE.93.042217}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2470-0045}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Phys. Rev. E}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">apr</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{4}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{042217}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Granger causality analysis with nonuniform sampling and its application to pulse-coupled nonlinear dynamics}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{93}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2016}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Barranca2016a" class="col-sm-11">
        <!-- Title -->
        <div class="title">Efficient image processing via compressive sensing of integrate-and-fire neuronal network dynamics</div>
        <!-- Author -->
        <div class="author">
        

        Barranca, Victor J., , Kovačič, Gregor, , <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>, and
          <span class="more-authors" title="click to view 1 more author" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '1 more author' ? 'David Cai' : '1 more author';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">1 more author</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Neurocomputing</em>, 171, 1313–1322 2016
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1016/j.neucom.2015.07.067"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1016/j.neucom.2015.07.067" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Integrate-and-fire (I&amp;F) neuronal networks are ubiquitous in diverse image processing applications, including image segmentation and visual perception. While conventional I&amp;F network image processing requires the number of nodes composing the network to be equal to the number of image pixels driving the network, we determine whether I&amp;F dynamics can accurately transmit image information when there are significantly fewer nodes than network input-signal components. Although compressive sensing (CS) theory facilitates the recovery of images using very few samples through linear signal processing, it does not address whether similar signal recovery techniques facilitate reconstructions through measurement of the nonlinear dynamics of an I&amp;F network. In this paper, we present a new framework for recovering sparse inputs of nonlinear neuronal networks via compressive sensing. By recovering both one-dimensional inputs and two-dimensional images, resembling natural stimuli, we demonstrate that input information can be well-preserved through nonlinear I&amp;F network dynamics even when the number of network-output measurements is significantly smaller than the number of input-signal components. This work suggests an important extension of CS theory potentially useful in improving the processing of medical or natural images through I&amp;F network dynamics and understanding the transmission of stimulus information across the visual system.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Barranca2016a</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Barranca, Victor J. and Kova{\v{c}}i{\v{c}}, Gregor and Zhou, Douglas and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.neucom.2015.07.067}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{18728286}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Neurocomputing}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Compressive sensing,Neuronal networks,Nonlinear dynamics,Signal processing}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1313--1322}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Efficient image processing via compressive sensing of integrate-and-fire neuronal network dynamics}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{171}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2016}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
</ol>

  <h2 class="year">2015</h2>
  <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Barranca2015a" class="col-sm-11">
        <!-- Title -->
        <div class="title">Low-rank network decomposition reveals structural characteristics of small-world networks</div>
        <!-- Author -->
        <div class="author">
        

        Barranca, Victor J., , <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>, and  and 
                Cai, David</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Phys. Rev. E</em>, 92(6), 062822 2015
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://link.aps.org/doi/10.1103/PhysRevE.92.062822" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1103/PhysRevE.92.062822"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1103/PhysRevE.92.062822" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Small-world networks occur naturally throughout biological, technological, and social systems. With their prevalence, it is particularly important to prudently identify small-world networks and further characterize their unique connection structure with respect to network function. In this work we develop a formalism for classifying networks and identifying small-world structure using a decomposition of network connectivity matrices into low-rank and sparse components, corresponding to connections within clusters of highly connected nodes and sparse interconnections between clusters, respectively. We show that the network decomposition is independent of node indexing and define associated bounded measures of connectivity structure, which provide insight into the clustering and regularity of network connections. While many existing network characterizations rely on constructing benchmark networks for comparison or fail to describe the structural properties of relatively densely connected networks, our classification relies only on the intrinsic network structure and is quite robust with respect to changes in connection density, producing stable results across network realizations. Using this framework, we analyze several real-world networks and reveal new structural properties, which are often indiscernible by previously established characterizations of network connectivity.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Barranca2015a</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Barranca, Victor J. and Zhou, Douglas and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1103/PhysRevE.92.062822}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1539-3755}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Phys. Rev. E}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">dec</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{6}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{062822}</span><span class="p">,</span>
  <span class="na">pmid</span> <span class="p">=</span> <span class="s">{26764759}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Low-rank network decomposition reveals structural characteristics of small-world networks}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{92}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2015}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Barranca2015" class="col-sm-11">
        <!-- Title -->
        <div class="title">A Novel Characterization of Amalgamated Networks in Natural Systems</div>
        <!-- Author -->
        <div class="author">
        

        Barranca, Victor J., , <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>, and  and 
                Cai, David</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Sci. Rep.</em>, 5(1), 10611 2015
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="http://dx.doi.org/10.1038/srep10611" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1038/srep10611"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1038/srep10611" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Densely-connected networks are prominent among natural systems, exhibiting structural characteristics often optimized for biological function. To reveal such features in highly-connected networks, we introduce a new network characterization determined by a decomposition of network-connectivity into low-rank and sparse components. Based on these components, we discover a new class of networks we define as amalgamated networks, which exhibit large functional groups and dense connectivity. Analyzing recent experimental findings on cerebral cortex, food-web, and gene regulatory networks, we establish the unique importance of amalgamated networks in fostering biologically advantageous properties, including rapid communication among nodes, structural stability under attacks, and separation of network activity into distinct functional modules. We further observe that our network characterization is scalable with network size and connectivity, thereby identifying robust features significant to diverse physical systems, which are typically undetectable by conventional characterizations of connectivity. We expect that studying the amalgamation properties of biological networks may offer new insights into understanding their structure-function relationships.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Barranca2015</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Barranca, Victor J. and Zhou, Douglas and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/srep10611}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2045-2322}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Sci. Rep.}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">sep</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{10611}</span><span class="p">,</span>
  <span class="na">pmid</span> <span class="p">=</span> <span class="s">{26035066}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Nature Publishing Group}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{A Novel Characterization of Amalgamated Networks in Natural Systems}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{5}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2015}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Li2015" class="col-sm-11">
        <!-- Title -->
        <div class="title">Analysis of the dendritic integration of excitatory and inhibitory inputs using cable models</div>
        <!-- Author -->
        <div class="author">
        

        <a href="https://ins.sjtu.edu.cn/people/songtingli/" rel="external nofollow noopener" target="_blank">Songting Li</a>, <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>, and  and 
                Cai, David</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Commun. Math. Sci.</em>, 13(2), 565–575 2015
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="http://www.intlpress.com/site/pub/pages/journals/items/cms/content/vols/0013/0002/a016/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.4310/CMS.2015.v13.n2.a16"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.4310/CMS.2015.v13.n2.a16" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>We address the question of how a neuron integrates excitatory (E) and inhibitory (I) synaptic inputs from different dendritic sites. For an idealized neuron model with an unbranched dendritic cable, we construct its Green’s function and carry out an asymptotic analysis to obtain its solutions. Using these asymptotic solutions, in the presence of E and I inputs, we can successfully reveal the underlying mechanisms of a dendritic integration rule, which was discovered in a recent experiment. Our analysis can be extended to the multi-branch case to characterize the E-I dendritic integration on any branches. The novel characterization is confirmed by the numerical simulation of a biologically realistic neuron.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Li2015</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Li, Songting and Zhou, Douglas and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.4310/CMS.2015.v13.n2.a16}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{15396746}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Commun. Math. Sci.}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Asymptotic solution,Cable model,Dendritic integration,Green's function}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{2}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{565--575}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Analysis of the dendritic integration of excitatory and inhibitory inputs using cable models}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{13}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2015}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
</ol>

  <h2 class="year">2014</h2>
  <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Barranca2014" class="col-sm-11">
        <!-- Title -->
        <div class="title">Network dynamics for optimal compressive-sensing input-signal recovery</div>
        <!-- Author -->
        <div class="author">
        

        Barranca, Victor J., , Kovačič, Gregor, , <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>, and
          <span class="more-authors" title="click to view 1 more author" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '1 more author' ? 'David Cai' : '1 more author';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">1 more author</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Phys. Rev. E</em>, 90(4), 042908 2014
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://link.aps.org/doi/10.1103/PhysRevE.90.042908" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1103/PhysRevE.90.042908"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1103/PhysRevE.90.042908" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>By using compressive sensing (CS) theory, a broad class of static signals can be reconstructed through a sequence of very few measurements in the framework of a linear system. For networks with nonlinear and time-evolving dynamics, is it similarly possible to recover an unknown input signal from only a small number of network output measurements? We address this question for pulse-coupled networks and investigate the network dynamics necessary for successful input signal recovery. Determining the specific network characteristics that correspond to a minimal input reconstruction error, we are able to achieve high-quality signal reconstructions with few measurements of network output. Using various measures to characterize dynamical properties of network output, we determine that networks with highly variable and aperiodic output can successfully encode network input information with high fidelity and achieve the most accurate CS input reconstructions. For time-varying inputs, we also find that high-quality reconstructions are achievable by measuring network output over a relatively short time window. Even when network inputs change with time, the same optimal choice of network characteristics and corresponding dynamics apply as in the case of static inputs.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Barranca2014</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Barranca, Victor J. and Kova{\v{c}}i{\v{c}}, Gregor and Zhou, Douglas and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1103/PhysRevE.90.042908}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1539-3755}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Phys. Rev. E}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">oct</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{4}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{042908}</span><span class="p">,</span>
  <span class="na">pmid</span> <span class="p">=</span> <span class="s">{25375568}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Network dynamics for optimal compressive-sensing input-signal recovery}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{90}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2014}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Zhang2014" class="col-sm-11">
        <!-- Title -->
        <div class="title">A coarse-grained framework for spiking neuronal networks: between homogeneity and synchrony</div>
        <!-- Author -->
        <div class="author">
        

        Zhang, Jiwei, , <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>, 
                Cai, David, and
          <span class="more-authors" title="click to view 1 more author" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '1 more author' ? 'Aaditya V. Rangan' : '1 more author';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">1 more author</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>J. Comput. Neurosci.</em>, 37(1), 81–104 2014
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="http://link.springer.com/10.1007/s10827-013-0488-y" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1007/s10827-013-0488-y"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1007/s10827-013-0488-y" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Homogeneously structured networks of neurons driven by noise can exhibit a broad range of dynamic behavior. This dynamic behavior can range from homogeneity to synchrony, and often incorporates brief spurts of collaborative activity which we call multiple-firing-events (MFEs). These multiple-firing- events depend on neither structured architecture nor structured input, and are an emergent property of the system. Although these MFEs likely play a major role in the neuronal avalanches observed in culture and in vivo, the mechanisms underlying these MFEs cannot easily be captured using current population-dynamics models. In this work we introduce a coarse-grained framework which illustrates certain dynamics responsible for the generation of MFEs. By using a new kind of ensemble-average, this coarse-grained framework can not only address the nucleation of MFEs, but can also faithfully capture a broad range of dynamic regimes ranging from homogeneity to synchrony. \textcopyright 2013 Springer Science+Business Media.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Zhang2014</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zhang, Jiwei and Zhou, Douglas and Cai, David and Rangan, Aaditya V.}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/s10827-013-0488-y}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0929-5313}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{J. Comput. Neurosci.}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Dynamical systems,Homogeneity,Multiple firing events,Partitioned-ensemble-average,Spiking neurons,Synchrony}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">aug</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{81--104}</span><span class="p">,</span>
  <span class="na">pmid</span> <span class="p">=</span> <span class="s">{24338105}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{A coarse-grained framework for spiking neuronal networks: between homogeneity and synchrony}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{37}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2014}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Barranca2014a" class="col-sm-11">
        <!-- Title -->
        <div class="title">Sparsity and Compressed Coding in Sensory Systems</div>
        <!-- Author -->
        <div class="author">
        

        Barranca, Victor J., , Kovačič, Gregor, , <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>, and
          <span class="more-authors" title="click to view 1 more author" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '1 more author' ? 'David Cai' : '1 more author';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">1 more author</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>PLoS Comput. Biol.</em>, 10(8), 1-11 2014
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://dx.plos.org/10.1371/journal.pcbi.1003793" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1371/journal.pcbi.1003793"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pcbi.1003793" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Considering that many natural stimuli are sparse, can a sensory system evolve to take advantage of this sparsity? We explore this question and show that significant downstream reductions in the numbers of neurons transmitting stimuli observed in early sensory pathways might be a consequence of this sparsity. First, we model an early sensory pathway using an idealized neuronal network comprised of receptors and downstream sensory neurons. Then, by revealing a linear structure intrinsic to neuronal network dynamics, our work points to a potential mechanism for transmitting sparse stimuli, related to compressed-sensing (CS) type data acquisition. Through simulation, we examine the characteristics of networks that are optimal in sparsity encoding, and the impact of localized receptive fields beyond conventional CS theory. The results of this work suggest a new network framework of signal sparsity, freeing the notion from any dependence on specific component-space representations. We expect our CS network mechanism to provide guidance for studying sparse stimulus transmission along realistic sensory pathways as well as engineering network designs that utilize sparsity encoding.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Barranca2014a</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Barranca, Victor J. and Kova{\v{c}}i{\v{c}}, Gregor and Zhou, Douglas and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1371/journal.pcbi.1003793}</span><span class="p">,</span>
  <span class="na">editor</span> <span class="p">=</span> <span class="s">{Sporns, Olaf}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1553-7358}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{PLoS Comput. Biol.}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">aug</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{8}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1-11}</span><span class="p">,</span>
  <span class="na">pmid</span> <span class="p">=</span> <span class="s">{25144745}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Sparsity and Compressed Coding in Sensory Systems}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{10}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2014}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Zhou2014" class="col-sm-11">
        <!-- Title -->
        <div class="title">Analysis of sampling artifacts on the Granger causality analysis for topology extraction of neuronal dynamics</div>
        <!-- Author -->
        <div class="author">
        

        <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>, Zhang, Yaoyu, , 
                Xiao, Yanyang, and
          <span class="more-authors" title="click to view 1 more author" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '1 more author' ? 'David Cai' : '1 more author';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">1 more author</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Front. Comput. Neurosci.</em>, 8, 75 2014
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="http://journal.frontiersin.org/article/10.3389/fncom.2014.00075/abstract" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.3389/fncom.2014.00075"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.3389/fncom.2014.00075" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Granger causality (GC) is a powerful method for causal inference for time series. In general, the GC value is computed using discrete time series sampled from continuous-time processes with a certain sampling interval length τ, i.e., the GC value is a function of τ. Using the GC analysis for the topology extraction of the simplest integrate-and-fire neuronal network of two neurons, we discuss behaviors of the GC value as a function of τ, which exhibits (i) oscillations, often vanishing at certain finite sampling interval lengths, (ii) the GC vanishes linearly as one uses finer and finer sampling. We show that these sampling effects can occur in both linear and non-linear dynamics: the GC value may vanish in the presence of true causal influence or become non-zero in the absence of causal influence. Without properly taking this issue into account, GC analysis may produce unreliable conclusions about causal influence when applied to empirical data. These sampling artifacts on the GC value greatly complicate the reliability of causal inference using the GC analysis, in general, and the validity of topology reconstruction for networks, in particular. We use idealized linear models to illustrate possible mechanisms underlying these phenomena and to gain insight into the general spectral structures that give rise to these sampling effects. Finally, we present an approach to circumvent these sampling artifacts to obtain reliable GC values. \textcopyright 2014 Zhou, Zhang, Xiao and Cai.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Zhou2014</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zhou, Douglas and Zhang, Yaoyu and Xiao, Yanyang and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.3389/fncom.2014.00075}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1662-5188}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Front. Comput. Neurosci.}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Causal inference,Granger causality,Neuronal dynamics,Reliability,Sampling rate,Topology extraction}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jul</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{75}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Analysis of sampling artifacts on the Granger causality analysis for topology extraction of neuronal dynamics}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{8}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2014}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Zhang2014a" class="col-sm-11">
        <!-- Title -->
        <div class="title">Distribution of correlated spiking events in a population-based approach for Integrate-and-Fire networks</div>
        <!-- Author -->
        <div class="author">
        

        Zhang, Jiwei, , Newhall, Katherine, , <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>, and
          <span class="more-authors" title="click to view 1 more author" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '1 more author' ? 'Aaditya Rangan' : '1 more author';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">1 more author</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>J. Comput. Neurosci.</em>, 36(2), 279–295 2014
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="http://link.springer.com/10.1007/s10827-013-0472-6" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1007/s10827-013-0472-6"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1007/s10827-013-0472-6" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Randomly connected populations of spiking neurons display a rich variety of dynamics. However, much of the current modeling and theoretical work has focused on two dynamical extremes: on one hand homogeneous dynamics characterized by weak correlations between neurons, and on the other hand total synchrony characterized by large populations firing in unison. In this paper we address the conceptual issue of how to mathematically characterize the partially synchronous "multiple firing events" (MFEs) which manifest in between these two dynamical extremes. We further develop a geometric method for obtaining the distribution of magnitudes of these MFEs by recasting the cascading firing event process as a first-passage time problem, and deriving an analytical approximation of the first passage time density valid for large neuron populations. Thus, we establish a direct link between the voltage distributions of excitatory and inhibitory neurons and the number of neurons firing in an MFE that can be easily integrated into population-based computational methods, thereby bridging the gap between homogeneous firing regimes and total synchrony. \textcopyright 2013 Springer Science+Business Media.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Zhang2014a</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zhang, Jiwei and Newhall, Katherine and Zhou, Douglas and Rangan, Aaditya}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/s10827-013-0472-6}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0929-5313}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{J. Comput. Neurosci.}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{First passage time,Homogeneity,Integrate and fire neuronal networks,Multiple firing events,Spiking neurons,Synchrony}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">apr</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{2}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{279--295}</span><span class="p">,</span>
  <span class="na">pmid</span> <span class="p">=</span> <span class="s">{23851661}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Distribution of correlated spiking events in a population-based approach for Integrate-and-Fire networks}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{36}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2014}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Zhou2014b" class="col-sm-11">
        <!-- Title -->
        <div class="title">Reliability of the Granger causality inference</div>
        <!-- Author -->
        <div class="author">
        

        <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>, Zhang, Yaoyu, , 
                Xiao, Yanyang, and
          <span class="more-authors" title="click to view 1 more author" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '1 more author' ? 'David Cai' : '1 more author';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">1 more author</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>New J. Phys.</em>, 16(4), 043016 2014
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://iopscience.iop.org/article/10.1088/1367-2630/16/4/043016" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1088/1367-2630/16/4/043016"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1088/1367-2630/16/4/043016" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>How to characterize information flows in physical, biological, and social systems remains a major theoretical challenge. Granger causality (GC) analysis has been widely used to investigate information flow through causal interactions. We address one of the central questions in GC analysis, that is, the reliability of the GC evaluation and its implications for the causal structures extracted by this analysis. Our work reveals that the manner in which a continuous dynamical process is projected or coarse-grained to a discrete process has a profound impact on the reliability of the GC inference, and different sampling may potentially yield completely opposite inferences. This inference hazard is present for both linear and nonlinear processes. We emphasize that there is a hazard of reaching incorrect conclusions about network topologies, even including statistical (such as small-world or scale-free) properties of the networks, when GC analysis is blindly applied to infer the network topology. We demonstrate this using a small-world network for which a drastic loss of small-world attributes occurs in the reconstructed network using the standard GC approach. We further show how to resolve the paradox that the GC analysis seemingly becomes less reliable when more information is incorporated using finer and finer sampling. Finally, we present strategies to overcome these inference artifacts in order to obtain a reliable GC result. \textcopyright 2014 IOP Publishing Ltd and Deutsche Physikalische Gesellschaft.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Zhou2014b</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zhou, Douglas and Zhang, Yaoyu and Xiao, Yanyang and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1088/1367-2630/16/4/043016}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1367-2630}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{New J. Phys.}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Causal information flow,Reliability of causal inference,Sampling hazards,Small-world topology}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">apr</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{4}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{043016}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{IOP Publishing}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Reliability of the Granger causality inference}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{16}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2014}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Zhou2014a" class="col-sm-11">
        <!-- Title -->
        <div class="title">Granger Causality Network Reconstruction of Conductance-Based Integrate-and-Fire Neuronal Systems</div>
        <!-- Author -->
        <div class="author">
        

        <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>, Xiao, Yanyang, , 
                Zhang, Yaoyu, and
          <span class="more-authors" title="click to view 2 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '2 more authors' ? 'Zhiqin Xu, David Cai' : '2 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">2 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>PLoS One</em>, 9(2), 1-17 2014
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://dx.plos.org/10.1371/journal.pone.0087636" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1371/journal.pone.0087636"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pone.0087636" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Reconstruction of anatomical connectivity from measured dynamical activities of coupled neurons is one of the fundamental issues in the understanding of structure-function relationship of neuronal circuitry. Many approaches have been developed to address this issue based on either electrical or metabolic data observed in experiment. The Granger causality (GC) analysis remains one of the major approaches to explore the dynamical causal connectivity among individual neurons or neuronal populations. However, it is yet to be clarified how such causal connectivity, i.e., the GC connectivity, can be mapped to the underlying anatomical connectivity in neuronal networks. We perform the GC analysis on the conductance-based integrate-and-fire (I&amp;F) neuronal networks to obtain their causal connectivity. Through numerical experiments, we find that the underlying synaptic connectivity amongst individual neurons or subnetworks, can be successfully reconstructed by the GC connectivity constructed from voltage time series. Furthermore, this reconstruction is insensitive to dynamical regimes and can be achieved without perturbing systems and prior knowledge of neuronal model parameters. Surprisingly, the synaptic connectivity can even be reconstructed by merely knowing the raster of systems, i.e., spike timing of neurons. Using spike-triggered correlation techniques, we establish a direct mapping between the causal connectivity and the synaptic connectivity for the conductance-based I&amp;F neuronal networks, and show the GC is quadratically related to the coupling strength. The theoretical approach we develop here may provide a framework for examining the validity of the GC analysis in other settings. \textcopyright 2014 Zhou et al.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Zhou2014a</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zhou, Douglas and Xiao, Yanyang and Zhang, Yaoyu and Xu, Zhiqin and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1371/journal.pone.0087636}</span><span class="p">,</span>
  <span class="na">editor</span> <span class="p">=</span> <span class="s">{Marinazzo, Daniele}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1932-6203}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{PLoS One}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">feb</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{2}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1-17}</span><span class="p">,</span>
  <span class="na">pmid</span> <span class="p">=</span> <span class="s">{24586285}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Granger Causality Network Reconstruction of Conductance-Based Integrate-and-Fire Neuronal Systems}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{9}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2014}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Li2014" class="col-sm-11">
        <!-- Title -->
        <div class="title">Bilinearity in Spatiotemporal Integration of Synaptic Inputs</div>
        <!-- Author -->
        <div class="author">
        

        <a href="https://ins.sjtu.edu.cn/people/songtingli/" rel="external nofollow noopener" target="_blank">Songting Li</a>, Liu, Nan, , 
                Zhang, Xiao, and
          <span class="more-authors" title="click to view 2 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '2 more authors' ? 'Douglas Zhou, David Cai' : '2 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">2 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>PLoS Comput. Biol.</em>, 10(12), 1-17 2014
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1371/journal.pcbi.1004014"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pcbi.1004014" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Neurons process information via integration of synaptic inputs from dendrites. Many experimental results demonstrate dendritic integration could be highly nonlinear, yet few theoretical analyses have been performed to obtain a precise quantitative characterization analytically. Based on asymptotic analysis of a two-compartment passive cable model, given a pair of time-dependent synaptic conductance inputs, we derive a bilinear spatiotemporal dendritic integration rule. The summed somatic potential can be well approximated by the linear summation of the two postsynaptic potentials elicited separately, plus a third additional bilinear term proportional to their product with a proportionality coefficient (Formula presented.) . The rule is valid for a pair of synaptic inputs of all types, including excitation-inhibition, excitation-excitation, and inhibition-inhibition. In addition, the rule is valid during the whole dendritic integration process for a pair of synaptic inputs with arbitrary input time differences and input locations. The coefficient (Formula presented.) is demonstrated to be nearly independent of the input strengths but is dependent on input times and input locations. This rule is then verified through simulation of a realistic pyramidal neuron model and in electrophysiological experiments of rat hippocampal CA1 neurons. The rule is further generalized to describe the spatiotemporal dendritic integration of multiple excitatory and inhibitory synaptic inputs. The integration of multiple inputs can be decomposed into the sum of all possible pairwise integration, where each paired integration obeys the bilinear rule. This decomposition leads to a graph representation of dendritic integration, which can be viewed as functionally sparse.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Li2014</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Li, Songting and Liu, Nan and hui Zhang, Xiao and Zhou, Douglas and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1371/journal.pcbi.1004014}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{15537358}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{PLoS Comput. Biol.}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{12}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Bilinearity in Spatiotemporal Integration of Synaptic Inputs}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{10}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2014}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
</ol>

  <h2 class="year">2013</h2>
  <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Zhou2013causal" class="col-sm-11">
        <!-- Title -->
        <div class="title">Causal and Structural Connectivity of Pulse-Coupled Nonlinear Networks</div>
        <!-- Author -->
        <div class="author">
        

        <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>, Xiao, Yanyang, , 
                Zhang, Yaoyu, and
          <span class="more-authors" title="click to view 2 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '2 more authors' ? 'Zhiqin Xu, David Cai' : '2 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">2 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Phys. Rev. Lett.</em>, 111(5), 054102 2013
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://link.aps.org/doi/10.1103/PhysRevLett.111.054102" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1103/PhysRevLett.111.054102"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1103/PhysRevLett.111.054102" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>We study the reconstruction of structural connectivity for a general class of pulse-coupled nonlinear networks and show that the reconstruction can be successfully achieved through linear Granger causality (GC) analysis. Using spike-triggered correlation of whitened signals, we obtain a quadratic relationship between GC and the network couplings, thus establishing a direct link between the causal connectivity and the structural connectivity within these networks. Our work may provide insight into the applicability of GC in the study of the function of general nonlinear networks. \textcopyright 2013 American Physical Society.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Zhou2013causal</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zhou, Douglas and Xiao, Yanyang and Zhang, Yaoyu and Xu, Zhiqin and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1103/PhysRevLett.111.054102}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0031-9007}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Phys. Rev. Lett.}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jul</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{5}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{054102}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{APS}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Causal and Structural Connectivity of Pulse-Coupled Nonlinear Networks}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{111}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2013}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Zhou2013a" class="col-sm-11">
        <!-- Title -->
        <div class="title">Spatiotemporal dynamics of neuronal population response in the primary visual cortex</div>
        <!-- Author -->
        <div class="author">
        

        <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>, Rangan, Aaditya V., , 
                McLaughlin, David W., and
          <span class="more-authors" title="click to view 1 more author" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '1 more author' ? 'David Cai' : '1 more author';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">1 more author</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Proc. Natl. Acad. Sci.</em>, 110(23), 9517–9522 2013
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://pnas.org/doi/full/10.1073/pnas.1308167110" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1073/pnas.1308167110"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1073/pnas.1308167110" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>One of the fundamental questions in system neuroscience is how the brain encodes external stimuli in the early sensory cortex. It has been found in experiments that even some simple sensory stimuli can activate large populations of neurons. It is believed that information can be encoded in the spatiotemporal profile of these collective neuronal responses. Here, we use a large-scale computational model of the primary visual cortex (V1) to study the population responses in V1 as observed in experiments in which monkeys performed visual detection tasks. We show that our model can capture very well spatiotemporal activities measured by voltage-sensitive-dye-based optical imaging in V1 of the awake state. In our model, the properties of horizontal long-range connections with NMDA conductance play an important role in the correlated population responses and have strong implications for spatiotemporal coding of neuronal populations. Our computational modeling approach allows us to reveal intrinsic cortical dynamics, separating them from those statistical effects arising from averaging procedures in experiment. For example, in experiments, it was shown that there was a spatially antagonistic center-surround structure in optimal weights in signal detection theory, which was believed to underlie the efficiency of population coding. However, our study shows that this feature is an artifact of data processing.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Zhou2013a</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zhou, Douglas and Rangan, Aaditya V. and McLaughlin, David W. and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1073/pnas.1308167110}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0027-8424}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Proc. Natl. Acad. Sci.}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Lateral long-range connection,Optimal detection theory,Pixel size,Population dynamics,Spatiotemporal patterns}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jun</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{23}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{9517--9522}</span><span class="p">,</span>
  <span class="na">pmid</span> <span class="p">=</span> <span class="s">{23696666}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Spatiotemporal dynamics of neuronal population response in the primary visual cortex}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{110}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2013}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Zhou2013phenomenological" class="col-sm-11">
        <!-- Title -->
        <div class="title">Phenomenological Incorporation of Nonlinear Dendritic Integration Using Integrate-and-Fire Neuronal Frameworks</div>
        <!-- Author -->
        <div class="author">
        

        <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>, <a href="https://ins.sjtu.edu.cn/people/songtingli/" rel="external nofollow noopener" target="_blank">Songting Li</a>, 
                Zhang, Xiao-hui, and
          <span class="more-authors" title="click to view 1 more author" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '1 more author' ? 'David Cai' : '1 more author';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">1 more author</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>PLoS One</em>, 8(1), 1-12 2013
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://dx.plos.org/10.1371/journal.pone.0053508" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1371/journal.pone.0053508"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pone.0053508" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>It has been discovered recently in experiments that the dendritic integration of excitatory glutamatergic inputs and inhibitory GABAergic inputs in hippocampus CA1 pyramidal neurons obeys a simple arithmetic rule as VSExp ≈ VEExp + VIExp + kVEExp VIExp, where VSExp, VEExp and VIExp are the respective voltage values of the summed somatic potential, the excitatory postsynaptic potential (EPSP) and the inhibitory postsynaptic potential measured at the time when the EPSP reaches its peak value. Moreover, the shunting coefficient k in this rule only depends on the spatial location but not the amplitude of the excitatory or inhibitory input on the dendrite. In this work, we address the theoretical issue of how much the above dendritic integration rule can be accounted for using subthreshold membrane potential dynamics in the soma as characterized by the conductance-based integrate-and-fire (I&amp;F) model. Then, we propose a simple I&amp;F neuron model that incorporates the spatial dependence of the shunting coefficient k by a phenomenological parametrization. Our analytical and numerical results show that this dendritic-integration-rule-based I&amp;F (DIF) model is able to capture many experimental observations and it also yields predictions that can be used to verify the validity of the DIF model experimentally. In addition, the DIF model incorporates the dendritic integration effects dynamically and is applicable to more general situations than those in experiments in which excitatory and inhibitory inputs occur simultaneously in time. Finally, we generalize the DIF neuronal model to incorporate multiple inputs and obtain a similar dendritic integration rule that is consistent with the results obtained by using a realistic neuronal model with multiple compartments. This generalized DIF model can potentially be used to study network dynamics that may involve effects arising from dendritic integrations. \textcopyright 2013 Zhou et al.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Zhou2013phenomenological</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zhou, Douglas and Li, Songting and Zhang, Xiao-hui and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1371/journal.pone.0053508}</span><span class="p">,</span>
  <span class="na">editor</span> <span class="p">=</span> <span class="s">{Chacron, Maurice J.}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1932-6203}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{PLoS One}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jan</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1-12}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Phenomenological Incorporation of Nonlinear Dendritic Integration Using Integrate-and-Fire Neuronal Frameworks}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{8}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2013}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
</ol>

  <h2 class="year">2012</h2>
  <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Sun2012" class="col-sm-11">
        <!-- Title -->
        <div class="title">Coarse-grained event tree analysis for quantifying Hodgkin-Huxley neuronal network dynamics</div>
        <!-- Author -->
        <div class="author">
        

        Sun, Yi, , Rangan, Aaditya V., , <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>, and
          <span class="more-authors" title="click to view 1 more author" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '1 more author' ? 'David Cai' : '1 more author';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">1 more author</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>J. Comput. Neurosci.</em>, 32(1), 55–72 2012
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="http://link.springer.com/10.1007/s10827-011-0339-7" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1007/s10827-011-0339-7"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1007/s10827-011-0339-7" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>We present an event tree analysis of studying the dynamics of the Hodgkin-Huxley (HH) neuronal networks. Our study relies on a coarse-grained projection to event trees and to the event chains that comprise these trees by using a statistical collection of spatial-temporal sequences of relevant physiological observables (such as sequences of spiking multiple neurons). This projection can retain information about network dynamics that covers multiple features, swiftly and robustly. We demonstrate that for even small differences in inputs, some dynamical regimes of HH networks contain sufficiently higher order statistics as reflected in event chains within the event tree analysis. Therefore, this analysis is effective in discriminating small differences in inputs. Moreover, we use event trees to analyze the results computed from an efficient library-based numerical method proposed in our previous work, where a pre-computed high resolution data library of typical neuronal trajectories during the interval of an action potential (spike) allows us to avoid resolving the spikes in detail. In this way, we can evolve the HH networks using time steps one order of magnitude larger than the typical time steps used for resolving the trajectories without the library, while achieving comparable statistical accuracy in terms of average firing rate and power spectra of voltage traces. Our numerical simulation results show that the library method is efficient in the sense that the results generated by using this numerical method with much larger time steps contain sufficiently high order statistical structure of firing events that are similar to the ones obtained using a regular HH solver. We use our event tree analysis to demonstrate these statistical similarities. \textcopyright 2011 Springer Science+Business Media, LLC.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Sun2012</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Sun, Yi and Rangan, Aaditya V. and Zhou, Douglas and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/s10827-011-0339-7}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0929-5313}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{J. Comput. Neurosci.}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Event tree analysis,Hodgkin-Huxley neuronal network,Information transmission,Library method,Neuronal coding}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">feb</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{55--72}</span><span class="p">,</span>
  <span class="na">pmid</span> <span class="p">=</span> <span class="s">{21597895}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Coarse-grained event tree analysis for quantifying Hodgkin-Huxley neuronal network dynamics}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{32}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2012}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li></ol>

  <h2 class="year">2010</h2>
  <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Zhou2010" class="col-sm-11">
        <!-- Title -->
        <div class="title">Spectrum of Lyapunov exponents of non-smooth dynamical systems of integrate-and-fire type</div>
        <!-- Author -->
        <div class="author">
        

        <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>, Sun, Yi, , 
                Rangan, Aaditya V., and
          <span class="more-authors" title="click to view 1 more author" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '1 more author' ? 'David Cai' : '1 more author';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">1 more author</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>J. Comput. Neurosci.</em>, 28(2), 229–245 2010
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="http://link.springer.com/10.1007/s10827-009-0201-3" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1007/s10827-009-0201-3"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1007/s10827-009-0201-3" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>We discuss how to characterize long-time dynamics of non-smooth dynamical systems, such as integrate-and-fire (I&amp;F) like neuronal network, using Lyapunov exponents and present a stable numerical method for the accurate evaluation of the spectrum of Lyapunov exponents for this large class of dynamics. These dynamics contain (i) jump conditions as in the firing-reset dynamics and (ii) degeneracy such as in the refractory period in which voltage-like variables of the network collapse to a single constant value. Using the networks of linear I&amp;F neurons, exponential I&amp;F neurons, and I&amp;F neurons with adaptive threshold, we illustrate our method and discuss the rich dynamics of these networks. \textcopyright Springer Science + Business Media, LLC 2009.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Zhou2010</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zhou, Douglas and Sun, Yi and Rangan, Aaditya V. and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/s10827-009-0201-3}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0929-5313}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{J. Comput. Neurosci.}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Firing-reset,Integrate-and-fire,Lyapunov exponents,Non-smooth,Refractory-induced degeneracy}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">apr</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{2}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{229--245}</span><span class="p">,</span>
  <span class="na">pmid</span> <span class="p">=</span> <span class="s">{20012178}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Spectrum of Lyapunov exponents of non-smooth dynamical systems of integrate-and-fire type}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{28}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2010}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="sun2010pseudo" class="col-sm-11">
        <!-- Title -->
        <div class="title">Pseudo-Lyapunov exponents and predictability of Hodgkin-Huxley neuronal network dynamics</div>
        <!-- Author -->
        <div class="author">
        

        Sun, Yi, , <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>, 
                Rangan, Aaditya V., and
          <span class="more-authors" title="click to view 1 more author" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '1 more author' ? 'David Cai' : '1 more author';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">1 more author</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>J. Comput. Neurosci.</em>, 28(2), 247–266 2010
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="http://link.springer.com/10.1007/s10827-009-0202-2" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1007/s10827-009-0202-2"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1007/s10827-009-0202-2" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>We present a numerical analysis of the dynamics of all-to-all coupled Hodgkin-Huxley (HH) neuronal networks with Poisson spike inputs. It is important to point out that, since the dynamical vector of the system contains discontinuous variables, we propose a so-called pseudo-Lyapunov exponent adapted from the classical definition using only continuous dynamical variables, and apply it in our numerical investigation. The numerical results of the largest Lyapunov exponent using this new definition are consistent with the dynamical regimes of the network. Three typical dynamical regimes - asynchronous, chaotic and synchronous, are found as the synaptic coupling strength increases from weak to strong. We use the pseudo-Lyapunov exponent and the power spectrum analysis of voltage traces to characterize the types of the network behavior. In the nonchaotic (asynchronous or synchronous) dynamical regimes, i.e., the weak or strong coupling limits, the pseudo-Lyapunov exponent is negative and there is a good numerical convergence of the solution in the trajectory-wise sense by using our numerical methods. Consequently, in these regimes the evolution of neuronal networks is reliable. For the chaotic dynamical regime with an intermediate strong coupling, the pseudo-Lyapunov exponent is positive, and there is no numerical convergence of the solution and only statistical quantifications of the numerical results are reliable. Finally, we present numerical evidence that the value of pseudo-Lyapunov exponent coincides with that of the standard Lyapunov exponent for systems we have been able to examine. \textcopyright Springer Science+Business Media, LLC 2009.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">sun2010pseudo</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Sun, Yi and Zhou, Douglas and Rangan, Aaditya V. and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/s10827-009-0202-2}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0929-5313}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{J. Comput. Neurosci.}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Chaos,Hodgkin-Huxley neuron,Lyapunov exponents,Neuronal network,Numerical analysis}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">apr</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{2}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{247--266}</span><span class="p">,</span>
  <span class="na">pmid</span> <span class="p">=</span> <span class="s">{20020192}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Springer}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Pseudo-Lyapunov exponents and predictability of Hodgkin-Huxley neuronal network dynamics}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{28}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2010}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Cai2010" class="col-sm-11">
        <!-- Title -->
        <div class="title">Dynamics of current-based, Poisson driven, integrate-and-fire neuronal networks</div>
        <!-- Author -->
        <div class="author">
        

        Cai, David, , Kovacic, Gregor, , 
                Kramer, Peter R., and
          <span class="more-authors" title="click to view 3 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '3 more authors' ? 'Katherine A. Newhall, Aaditya V. Rangan, Douglas Zhou' : '3 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">3 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Commun. Math. Sci.</em>, 8(2), 541–600 2010
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="http://www.intlpress.com/site/pub/pages/journals/items/cms/content/vols/0008/0002/a012/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.4310/CMS.2010.v8.n2.a12"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.4310/CMS.2010.v8.n2.a12" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Synchronous and asynchronous dynamics in all-to-all coupled networks of identical, excitatory, current-based, integrate-and-fire (I&amp;F) neurons with delta-impulse coupling currents and Poisson spike-train external drive are studied. Repeating synchronous total firing events, during which all the neurons fire simultaneously, are observed using numerical simulations and found to be the attracting state of the network for a large range of parameters. Mechanisms leading to such events are then described in two regimes of external drive: superthreshold and subthreshold. In the former, a probabilistic argument similar to the proof of the Central Limit Theorem yields the oscillation period, while in the latter, this period is analyzed via an exit time calculation utilizing a diffusion approximation of the Kolmogorov forward equation. Asynchronous dynamics are observed computationally in networks with random transmission delays. Neuronal voltage probability density functions (PDFs) and gain curves-graphs depicting the dependence of the network firing rate on the external drive strength-are analyzed using the steady solutions of the self-consistency problem for a Kolmogorov forward equation. All the voltage PDFs are obtained analytically, and asymptotic solutions for the gain curves are obtained in several physiologically relevant limits. The absence of chaotic dynamics is proved for the type of network under investigation by demonstrating convergence in time of its trajectories. \textcopyright 2010 International Press.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Cai2010</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Cai, David and Kovacic, Gregor and Kramer, Peter R. and Newhall, Katherine A. and Rangan, Aaditya V. and Zhou, Douglas}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.4310/CMS.2010.v8.n2.a12}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{15396746}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Commun. Math. Sci.}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Chaos,Exit-time,Neuronal network,Synchrony}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{2}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{541--600}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Dynamics of current-based, Poisson driven, integrate-and-fire neuronal networks}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{8}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2010}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
</ol>

  <h2 class="year">2009</h2>
  <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Sun2009" class="col-sm-11">
        <!-- Title -->
        <div class="title">Library-based numerical reduction of the Hodgkin–Huxley neuron for network simulation</div>
        <!-- Author -->
        <div class="author">
        

        Sun, Yi, , <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>, 
                Rangan, Aaditya V., and
          <span class="more-authors" title="click to view 1 more author" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '1 more author' ? 'David Cai' : '1 more author';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">1 more author</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>J. Comput. Neurosci.</em>, 27(3), 369–390 2009
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="http://link.springer.com/10.1007/s10827-009-0151-9" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1007/s10827-009-0151-9"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1007/s10827-009-0151-9" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>We present an efficient library-based numerical method for simulating the Hodgkin - Huxley (HH) neuronal networks. The key components in our numerical method involve (i) a pre-computed high resolution data library which contains typical neuronal trajectories (i.e., the time-courses of membrane potential and gating variables) during the interval of an action potential (spike), thus allowing us to avoid resolving the spikes in detail and to use large numerical time steps for evolving the HH neuron equations; (ii) an algorithm of spike-spike corrections within the groups of strongly coupled neurons to account for spike-spike interactions in a single large time step. By using the library method, we can evolve the HH networks using time steps one order of magnitude larger than the typical time steps used for resolving the trajectories without the library, while achieving comparable resolution in statistical quantifications of the network activity, such as average firing rate, interspike interval distribution, power spectra of voltage traces. Moreover, our large time steps using the library method can break the stability requirement of standard methods (such as Runge-Kutta (RK) methods) for the original dynamics. We compare our library-based method with RK methods, and find that our method can capture very well phase-locked, synchronous, and chaotic dynamics of HH neuronal networks. It is important to point out that, in essence, our library-based HH neuron solver can be viewed as a numerical reduction of the HH neuron to an integrate-and-fire (I&amp;F) neuronal representation that does not sacrifice the gating dynamics (as normally done in the analytical reduction to an I&amp;F neuron). \textcopyright Springer Science+Business Media, LLC 2009.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Sun2009</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Sun, Yi and Zhou, Douglas and Rangan, Aaditya V. and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/s10827-009-0151-9}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0929-5313}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{J. Comput. Neurosci.}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Chaos,Hodgkin - Huxley neuron,Integrate-and-fire neurons,Library method,Neuronal network,Numerical algorithm}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">dec</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{3}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{369--390}</span><span class="p">,</span>
  <span class="na">pmid</span> <span class="p">=</span> <span class="s">{19401809}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Library-based numerical reduction of the Hodgkin–Huxley neuron for network simulation}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{27}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2009}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <!-- <div class="col-sm-2 abbr"></div> -->

        <!-- Entry bib key -->
        <div id="Zhou2009" class="col-sm-11">
        <!-- Title -->
        <div class="title">Network-induced chaos in integrate-and-fire neuronal ensembles</div>
        <!-- Author -->
        <div class="author">
        

        <a href="https://ins.sjtu.edu.cn/people/zdz/" rel="external nofollow noopener" target="_blank">Douglas Zhou</a>, Rangan, Aaditya V., , 
                Sun, Yi, and
          <span class="more-authors" title="click to view 1 more author" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '1 more author' ? 'David Cai' : '1 more author';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">1 more author</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Phys. Rev. E</em>, 80(3), 031918 2009
        </div>
        <div class="periodical">
          
        </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://link.aps.org/doi/10.1103/PhysRevE.80.031918" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1103/PhysRevE.80.031918"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1103/PhysRevE.80.031918" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>It has been shown that a single standard linear integrate-and-fire (IF) neuron under a general time-dependent stimulus cannot possess chaotic dynamics despite the firing-reset discontinuity. Here we address the issue of whether conductance-based, pulsed-coupled network interactions can induce chaos in an IF neuronal ensemble. Using numerical methods, we demonstrate that all-to-all, homogeneously pulse-coupled IF neuronal networks can indeed give rise to chaotic dynamics under an external periodic current drive. We also provide a precise characterization of the largest Lyapunov exponent for these high dimensional nonsmooth dynamical systems. In addition, we present a stable and accurate numerical algorithm for evaluating the largest Lyapunov exponent, which can overcome difficulties encountered by traditional methods for these nonsmooth dynamical systems with degeneracy induced by, e.g., refractoriness of neurons. \textcopyright 2009 The American Physical Society.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Zhou2009</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zhou, Douglas and Rangan, Aaditya V. and Sun, Yi and Cai, David}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1103/PhysRevE.80.031918}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1539-3755}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Phys. Rev. E}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">sep</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{3}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{031918}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Network-induced chaos in integrate-and-fire neuronal ensembles}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{80}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2009}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
</ol>


</div>

          </article>

        </div>

    </div>

    <!-- Footer -->    
    <footer class="fixed-bottom">
      <div class="container mt-0">
        © Copyright 2024 Lab of Computational Neuroscience  . Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>.

      </div>
    </footer>

    <!-- JavaScripts -->
    <!-- jQuery -->
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.6.0/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>

    <!-- Bootsrap & MDB scripts -->
  <script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap/4.6.1/js/bootstrap.bundle.min.js" integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script>

    <!-- Masonry & imagesLoaded -->
  <script defer src="https://cdnjs.cloudflare.com/ajax/libs/masonry/4.2.2/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script>
  <script defer src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.0.0/imagesloaded.pkgd.min.js"></script>
  <script defer src="/assets/js/masonry.js" type="text/javascript"></script>
    
  <!-- Medium Zoom JS -->
  <script defer src="https://cdnjs.cloudflare.com/ajax/libs/medium-zoom/1.0.8/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script>
  <script defer src="/assets/js/zoom.js"></script><!-- Load Common JS -->
  <script defer src="/assets/js/common.js"></script>
  <script defer src="/assets/js/copy_code.js" type="text/javascript"></script>

    
  <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>
  <script async src="https://badge.dimensions.ai/badge.js"></script>

    <!-- MathJax -->
  <script type="text/javascript">
    window.MathJax = {
      tex: {
        tags: 'ams'
      }
    };
  </script>
  <script defer type="text/javascript" id="MathJax-script" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/3.2.0/es5/tex-mml-chtml.js"></script>
  <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>

    <!-- Global site tag (gtag.js) - Google Analytics -->
  <script async src="https://www.googletagmanager.com/gtag/js?id=G-KYK03DL0RL"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){ window.dataLayer.push(arguments); }
    gtag('js', new Date());
    gtag('config', 'G-KYK03DL0RL');
  </script>
    

<!-- Scrolling Progress Bar -->
<script type="text/javascript">
  /*
   * This JavaScript code has been adapted from the article 
   * https://css-tricks.com/reading-position-indicator/ authored by Pankaj Parashar, 
   * published on the website https://css-tricks.com on the 7th of May, 2014.
   * Couple of changes were made to the original code to make it compatible 
   * with the `al-foio` theme.
   */
  const progressBar = $("#progress");
  /*
   * We set up the bar after all elements are done loading.
   * In some cases, if the images in the page are larger than the intended
   * size they'll have on the page, they'll be resized via CSS to accomodate
   * the desired size. This mistake, however, breaks the computations as the
   * scroll size is computed as soon as the elements finish loading.
   * To account for this, a minimal delay was introduced before computing the
   * values.
   */
  window.onload = function () {
    setTimeout(progressBarSetup, 50);
  };
  /*
   * We set up the bar according to the browser.
   * If the browser supports the progress element we use that.
   * Otherwise, we resize the bar thru CSS styling
   */
  function progressBarSetup() {
    if ("max" in document.createElement("progress")) {
      initializeProgressElement();
      $(document).on("scroll", function() {
        progressBar.attr({ value: getCurrentScrollPosition() });
      });
      $(window).on("resize", initializeProgressElement);
    } else {
      resizeProgressBar();
      $(document).on("scroll", resizeProgressBar);
      $(window).on("resize", resizeProgressBar);
    }
  }
  /*
   * The vertical scroll position is the same as the number of pixels that
   * are hidden from view above the scrollable area. Thus, a value > 0 is
   * how much the user has scrolled from the top
   */
  function getCurrentScrollPosition() {
    return $(window).scrollTop();
  }

  function initializeProgressElement() {
    let navbarHeight = $("#navbar").outerHeight(true);
    $("body").css({ "padding-top": navbarHeight });
    $("progress-container").css({ "padding-top": navbarHeight });
    progressBar.css({ top: navbarHeight });
    progressBar.attr({
      max: getDistanceToScroll(),
      value: getCurrentScrollPosition(),
    });
  }
  /*
   * The offset between the html document height and the browser viewport
   * height will be greater than zero if vertical scroll is possible.
   * This is the distance the user can scroll
   */
  function getDistanceToScroll() {
    return $(document).height() - $(window).height();
  }

  function resizeProgressBar() {
    progressBar.css({ width: getWidthPercentage() + "%" });
  }
  // The scroll ratio equals the percentage to resize the bar
  function getWidthPercentage() {
    return (getCurrentScrollPosition() / getDistanceToScroll()) * 100;
  }
</script>

  </body>
</html>
